let ntp = {
    //insert key:value pairs refrerenced in html file
    normal: "No targeted rearrangements are identified.   About 19 % of lymphoblastic leukemia / lymphoma cases have clinically relevant abnormalities not detected by conventional karyotyping or routine clinical FISH assays, including BCR - ABL1 - like(Ph - like) B - lymphoblastic leukemia / lymphoma, which may be detected by next generation sequencing  or with probes in the ALL FISH Panel (Ph - Like).",
    Phpositive: "BCR::ABL1 gene fusion is observed in 20 % of adult B - ALL and is associated with an unfavorable prognosis.However, treatment with tyrosine kinase inhibitors has improved outcome.  If clinically indicated, molecular studies should be performed for characterization of the BCR::ABL1 fusion to aid in distinguishing between fusion genes p210, p230 and p190.  Molecular studies also help to establish baseline for future monitoring of therapy.",
    Ph2R1G1F: "The 2R2A1G1F signal pattern is suggestive of a clone characterized by insertion of material from the long arm of chromosome 9 into the long arm of chromosome 22, resulting in a BCR::ABL1 fusion gene.  BCR::ABL1 gene fusion is observed in 20 % of adult B - ALL and is associated with an unfavorable prognosis. However, treatment with tyrosine kinase inhibitors has improved outcome.  Cryptic rearrangements, such as insertions, are not detectable by conventional cytogenetics.  If clinically indicated, molecular studies should be performed for characterization of the BCR:: ABL1 fusion to aid in distinguishing between fusion genes p210, p230 and p190.  Molecular studies also help to establish baseline for future monitoring of therapy.  These results should be interpreted in conjunction with clinical and other laboratory findings.Monitoring by cytogenetics, FISH and / or molecular studies is recommended.<br>",
    Ph1R2G1F: "The 1R2A2G1F signal pattern is suggestive of a clone characterized by insertion of material from the long arm of chromosome 22 into the long arm of chromosome 9, resulting in a BCR::ABL1 fusion gene.  BCR::ABL1 gene fusion is observed in 20 % of adult B - ALL and is associated with an unfavorable prognosis. However, treatment with tyrosine kinase inhibitors has improved outcome.  Cryptic rearrangements, such as insertions, are not detectable by conventional cytogenetics.  If clinically indicated, molecular studies should be performed for characterization of the BCR:: ABL1 fusion to aid in distinguishing between fusion genes p210, p230 and p190.  Molecular studies also help to establish baseline for future monitoring of therapy.  These results should be interpreted in conjunction with clinical and other laboratory findings.Monitoring by cytogenetics, FISH and / or molecular studies is recommended.<br>",
    Ph2R2G1F: "The 2R2A2G1F signal pattern is suggestive of a clone characterized by a three-way translocation resulting in the der(22)t(9;22) [the Philadelphia chromosome] - a variant of the t(9;22)(q34.1;q11.2) - resulting in a BCR::ABL1 fusion gene.  BCR::ABL1 gene fusion is observed in 20 % of adult B - ALL and is associated with an unfavorable prognosis. However, treatment with tyrosine kinase inhibitors has improved outcome.  Cryptic rearrangements, such as insertions, are not detectable by conventional cytogenetics.  If clinically indicated, molecular studies should be performed for characterization of the BCR:: ABL1 fusion to aid in distinguishing between fusion genes p210, p230 and p190.  Molecular studies also help to establish baseline for future monitoring of therapy.  These results should be interpreted in conjunction with clinical and other laboratory findings.Monitoring by cytogenetics, FISH and / or molecular studies is recommended.<br>",
    Ph1R1G3F: "The 1R1G3F signal pattern is suggestive of the presence of the t(9;22) and an additional copy of the Philadelphia chromosome [der(22)t(9;22)]. BCR::ABL1 gene fusion is observed in 20 % of adult B - ALL and is associated with an unfavorable prognosis. However, treatment with tyrosine kinase inhibitors has improved outcome.  Cryptic rearrangements, such as insertions, are not detectable by conventional cytogenetics.  If clinically indicated, molecular studies should be performed for characterization of the BCR:: ABL1 fusion to aid in distinguishing between fusion genes p210, p230 and p190.  Molecular studies also help to establish baseline for future monitoring of therapy.  These results should be interpreted in conjunction with clinical and other laboratory findings.Monitoring by cytogenetics, FISH and / or molecular studies is recommended.<br>",
    PhHyperdiploid: "Additional copies of chromosomes -- and -- are observed in ALL characterized by hyperdiploidy.  However, in this case, these abnormalities likely represent secondary changes, in the primary clone characterized by the t(9;22) (34.1;q11.2) and may be indicative of clonal evolution.  The low risk associated with classic hyperdiploidy is not applicable in cases with concurrent presence of the t(9; 22).  BCR::ABL1 gene fusion is observed in 20 % of adult B - ALL and is associated with an unfavorable prognosis.However treatment with tyrosine kinase inhibitors has improved outcome.  The additional gains in signal patterns may represent abnormal clones, resulting from clonal evolution of the abnormal clone characterized by the 1R1G2F2A signal pattern.  The presence of other abnormalities, not detectable by this FISH probe set, cannot be ruled out.  If clinically indicated, molecular studies should be performed for characterization of the BCR::ABL1 fusion to aid in distinguishing between fusion genes p210, p230 and p190.  Molecular studies also help to establish baseline for future monitoring of therapy.  These results should be interpreted in conjunction with clinical and other laboratory findings.Monitoring by cytogenetics, FISH and / or molecular studies is recommended.",
    Ph3R3A: "The 3R3A signal pattern is consistent with gain of chromosome 9 or the long arm of chromosome 9. Trisomy 9 is a recurring abnormality in MDS, MPN and AML. Among MPN, it is observed in 7 % of all polycythemia vera cases.",
    CDKN2A1R2G: "Loss of the CDKN2A gene is frequently observed in acute lymphoblastic leukemia and is associated with an adverse prognosis.",
    CDKN2A1R1G: "Loss of the CDKN2A gene is frequently observed in acute lymphoblastic leukemia and is associated with an adverse prognosis.Although monosomy 9 includes loss of material from the short arm of chromosome 9, it is unclear if this has the same prognostic significance as described in patients with CDKN2A deletion.",
    trisomy10: "Trisomy 10 has been reported as a rare abnormality in B - ALL.   Hyperdiploidy in B - ALL tends to show a pattern of chromosome gain with extra copies of chromosomes 4, 6, 10, 14, 17, 18, 21 and X. In this case, the signal pattern for chromosomes 4, 6 and 17 do not indicate presence of more than two copies of each of the chromosomes.",
    hyperdiploidy: "Additional copies of chromosomes X, 4, 6, 10, 14, 17, 18 and 21 are observed in varying combinations and frequencies in acute lymphoblastic leukemia (ALL) characterized by high-hyperdiploidy.  Inadequate data is available to determine the prognostic significance of high-hyperdiploidy in adults.",
    hyperdiploidyAtypical: "Hyperdiploidy in B-ALL tends to show a pattern of chromosome gain with extra copies of chromosomes 4, 6, 10, 14, 17, 18, 21 and X.  In this case, the signal pattern is suggestive of gain of most chromosomes analyzed using the FISH panel. Therefore it is not certain if these results represent the typical  hyperdiploidy observed in B-ALL. ",
    hyperdiploidyTetraploidy: "Hyperdiploidy in B-ALL, more often results in trisomy rather than tetrasomy for the gained chromosomes and tends to show a pattern of chromosome gain with extra copies of chromosomes 4, 6, 10, 14, 17, 18, 21 and X in varying combinations and frequencies.  In this case, some chromosomes are represented by three copies (see summary table, above) and some chromosomes  are represented by four copies. Therefore, it is not certain whether this karyotype represents the typical high hyperdiploidy observed in B-ALL or tetraploidization of a hypodiploid clone with subsequent loss of some chromosomes.",
    hypodiploidy: "Hypodiploidy is observed in B-ALL and is associated with unfavorable prognosis.",
    lowHypodiploidy: "Hypodiploidy is observed in B-ALL and is associated with unfavorable prognosis. The signal pattern observed in this case is suggestive of low hypodiploidy.  If clinically indicated SNP array studies to evaluate for loss of heterozygosity for confirmation  may be considered.",
    nearHaploidy: "Hypodiploidy is observed in B-ALL and is associated with unfavorable prognosis. The signal patterns observed in this case are suggestive of near haploidy with doubling of the complement, a well-described phenomenon.  If clinically indicated SNP array studies to evaluate for loss of heterozygosity may be considered to investigate this possibility further.",
    IGH1R1G1F: "Rearrangements involving the IGH locus are observed in approximately 5% of acute lymphoblastic leukemia and are associated with unfavorable prognosis. Translocation partners include CRLF2, EPOR, DUX4, IL3 and five members of the CEBP gene family: CEBPA (19q13), CEBPB (20q13), CEBPD (8q11), CEBPE (14q11), and CEBPG (19q13).",
    KMT2A: "KMT2A rearrangements are associated with unfavorable prognosis in B-ALL.",
    KMT2A1R1F: "The 1R1F signal pattern may be indicative of an unbalanced rearrangement involving KMT2A. KMT2A rearrangements are associated with unfavorable prognosis.",
    KMT2A1F: "The 1F signal pattern with the KMT2A probe is suggestive of loss of one copy of chromosome 11 or deletion of the long arm of chromosome 11. Loss of material from the long arm of chromosome 11 has been observed in myeloid (MDS) as well as lymphoid disorders (CLL).  Its presence in ALL is of uncertain significance.",
    KMT2A3F: "The 3F signal pattern with the KMT2A probe  is suggestive of gain of chromosome 11 or the long arm of chromosome 11. Gain of the long arm of chromosome 11, including band q23, is a recurring abnormality in MDS and AML. It is generally associated with unfavorable prognosis.  Its presence in ALL is of uncertain signficance.",
    MYC: "Rearrangements of the MYC gene are recurring abnormalities in lymphoid disorders, including B - lymphoblastic leukemia / lymphoma.",
    PBX1: "The t(1;19) (q23;p13.3) is observed in 6 % of adult B - ALL, and  is associated with intermediate risk. Historically the t(1;19) was associated with an unfavorable prognosis; however, it may have less impact on outcome with contemporary chemotherapy protocols.",
    PBX12R1G1F: "TCF3::PBX1 rearrangement may occur as a balanced translocation, t(1;19)(q23;p13), or as an unbalanced translocation, der(19)t(1;19)(q23;p13), in which only the derivative chromosome 19 is present. The unbalanced translocation, der(19)t(1;19), represented by the signal pattern observed here, is the more common form and accounts for 75% of all TCF3(E2A)::PBX1 rearrangements.  Both balanced and unbalanced translocations are sometimes observed in the same patient as separate clones. The TCF3(E2A)::PBX1 t(1;19) gene fusion is observed in 6% of adult B-ALL, and  is associated with intermediate risk. Historically the t(1;19) was associated with an unfavorable prognosis; however, it may have less impact on outcome with contemporary chemotherapy protocols.",
    TCF31G: "Submicroscopic deletions of TCF3 have been observed in B-ALL.",
    other: "The presence of other abnormalities, not detectable by this FISH probe set, cannot be ruled out.",
    conjunction: "These results should be interpreted in conjunction with clinical and other laboratory findings. Monitoring by cytogenetics and FISH studies is recommended."
};

let references = {

    //insert keys for reference list, below
        normal: ["Wang", "Tasian"],
        Phpositive: ["Moorman"],
        Ph2R1G1F: ["Moorman"],
        Ph1R2G1F: ["Moorman"],
        Ph2R2G1F: ["Moorman"],
        Ph1R1G3F: ["Moorman"],
        PhHyperdiploid: ["Moorman"],
        Ph3R3A: ["Haferlach", "Vandenberghe"],
        CDKN2A1R2G: ["Carter", "Ribera", "Braun", "Messina"],
        CDKN2A1R1G: ["Carter", "Ribera", "Braun", "Messina"],
        trisomy10: ["Okamura"],
        hyperdiploidy: ["Moorman", "Swerdlow"],
        hypodiploidy: ["Moorman2"],
        lowHypodiploidy: ["Moorman2", "Safavi", "Carroll"],
        nearHaploidy: ["Moorman2", "Safavi", "Carroll"],
        IGH1R1G1F: ["Moorman", "Schwab"],
        KMT2A: ["Moorman"],
        KMT2A1R1F: ["Moorman", "Schwab"],
        KMT2A3F: ["Sarova", "Andersen", "Wang", "Alseraye"],
        MYC: ["Li"],
        PBX1: ["Moorman"],
        PBX12R1G1F: ["Moorman"],
        TCF31G: ["Barber"],


        //create blank key if there are no references for the comment
        indeterminate: "",
        hyperdiploidyAtypical: "",
        hyperdiploidyTetraploidy: "",
        KMT2A1F: "",
        other: "",
        conjunction: "",
};

let reference_list = {

    //list all references
    Wang: "Wang Y, Miller S, Roulston D, et al. Genome - Wide Single - Nucleotide Polymorphism Array Analysis Improves Prognostication of Acute Lymphoblastic Leukemia / Lymphoma.Mol Diagn. 2016; 18(4): 595 - 603.<br>",
    Tasian: "Tasian SK, Loh ML and Hunger SP. Philadelphia chromosome–like acute lymphoblastic leukemia.Blood. 2017; 130: 2064 - 2072.<br>",
    Moorman: "Moorman AV. New and emerging prognostic and predictive genetic biomarkers in B - cell precursor acute lymphoblastic leukemia.Haematologica. 2016 Apr; 101(4): 407 - 16.<br>",
    Moorman2: "Moorman AV. The clinical relevance of chromosomal and genomic abnormalities in B-cell precursor acute lymphoblastic leukaemia. Blood Rev. 2012;26(3):123-35.<br>",
    Haferlach: "Haferlach, C and Bacher, U.   + 9 or trisomy 9   Atlas Genet Cytogenet Oncol Haematol. 2007; 11(1): 40 - 41.  http://AtlasGeneticsOncology.org/Anomalies/tri9ID1020.html <br>",
    Vandenberghe: "Vandenberghe P et al. Chronic Myeloproliferative Neoplasms.In Cancer Cytogenetics.Third Edition.Ed.Heim S and Mitelman F.Published by Wiley - Blackewell.  2009. Pages 209 - 232.<br>",
    Carter: "Carter TL, et al. Hemizygous p16(INK4A) deletion in pediatric acute lymphoblastic leukemia predicts independent risk of relapse.Blood. 2001; 97: 572 - 574.<br>",
    Ribera: "Ribera J, et al. Prognostic significance of copy number alterations in adolescent and adult patients with precursor B acute lymphoblastic leukemia enrolled in PETHEMA protocols.Cancer. 2015; 121: 3809 - 3817.<br>",
    Braun: "Braun M, et al. Biallelic loss of CDKN2A is associated with poor response to treatment in pediatric acute lymphoblastic leukemia.Leuk Lymphoma. 2017; 58: 1162 - 1171.<br>",
    Messina: "Messina M, et al. Clinical significance of recurrent copy number aberrations in B - lineage acute lymphoblastic leukemia without recurrent fusion genes across age cohorts.Br J Haematol.  2017; 178: 583 - 587.<br>",
    Okamura: "Okamura A et al. Trisomy 10 in adult pre- B - cell leukemia.Am J Hematol. 1999  May; 61(1): 83. PubMed PMID: 10331517.<br>",
    Swerdlow: "Swerdlow SH et al. (Eds): WHO Classification of Tumors of Haematopoietic and Lymphoid Tissues. (Revised 4th edition).IARC: Lyon 2017. Pp 205-206.<br>",
    Safavi: "Safavi S et al. Near - haploid and low- hypodiploid acute lymphoblastic leukemia: two distinct subtypes with consistently poor prognosis.Blood. 2017 Jan 26; 129(4): 420 - 423.<br>",
    Carroll: "Carroll AJ et al. Masked hypodiploidy: Hypodiploid acute lymphoblastic leukemia(ALL) mimicking hyperdiploid ALL in children: A report from the Children's Oncology Group. Cancer Genet. 2019 Oct;238:62-68. doi: 10.1016/j.cancergen.2019.07.009. Epub 2019 Jul 30.<br>",
    Schwab: "Schwab C, Harrison CJ. Advances in B - cell Precursor Acute Lymphoblastic Leukemia Genomics.Hemasphere. 2018 Jun 20; 2(4): e53.<br>",
    Sarova: "Sarova I et al. Cytogenetic manifestation of chromosome 11 duplication / amplification in acute myeloid leukemia.Cancer Genet Cytogenet. 199: 121 - 7, 2010.<br>",
    Andersen: "Andersen MK et al. Duplication or amplification of chromosome band 11q23, including the unrearranged MLL gene, is a recurrent abnormality in therapy - related MDS and AML, and is closely related to mutation of the TP53 gene and to previous therapy with alkylating agents.Genes Chromosomes Cancer. 31: 33 - 41, 2001.<br>",
    Wang: "Wang SA et al.Trisomy 11 in myelodysplastic syndromes defines a unique group of disease with aggressive clinicopathologic features.Leukemia. 24: 740 - 747, 2010.<br>",
    Alseraye: "Alseraye FM et al.Trisomy 11 as an isolated abnormality in acute myeloid leukemia is associated with unfavorable prognosis but not with an NPM1 or KIT mutation.Int J Clin Exp Pathol.4: 371 - 377, 2011.<br>",
    Li: "Li Y et al. B Lymphoblastic Leukemia / Lymphoma With Burkitt - like Morphology and IGH / MYC Rearrangement: Report of 3 Cases in Adult Patients.Am J Surg Pathol. 2018 Feb; 42(2): 269 - 276.<br>",
    Barber: " Barber KE et al.Molecular cytogenetic characterization of TCF3(E2A)/ 19p13.3 rearrangements in B - cell precursor acute lymphoblastic leukemia.Genes Chromosomes Cancer. 2007; 46(5): 478 - 486.<br>"



};


//do not change anything below this line

let reference_array = []; //to hold all reference keys
let ref_unique_array = []; //only unique reference keys

function detectedAbnormality(ntpKey, reference) {
    $("<span>" + ntpKey + "<br>" + "<br>" + "</span>").appendTo("#allntp");
    //write the same reference only once
    for (let el of reference) {
        reference_array.push(el);
    }
    ref_unique_array = [...new Set(reference_array)];
    let final_list = []; //to hold all unique reference values; local variable required else list will grow with each write
    for (let el of ref_unique_array) {
        final_list.push(reference_list[el]);
    }
    final_list_string = final_list.toString().replace(/<br>,/g, "<br>"); //regex to stop printing of comma before references
    document.querySelector("#references").innerHTML = final_list_string;
}

function copyToClipboard() {
    var copyText =
        document.getElementById("reportText").innerText +
        document.getElementById("allReferences").innerText;
    navigator.clipboard.writeText(copyText);
    console.log("Text copied");
}

function copyInterpToClipboard() {
    var copyText = document.querySelector("#reportText").innerText;
    navigator.clipboard.writeText(copyText);
}

function copyRefsToClipboard() {
    var copyText = document.querySelector("#references").innerText;
    navigator.clipboard.writeText(copyText);
}
