let dataApvx = {
"ABL1":	"ABL1, located on 9q34.1, encodes for a tyrosine kinase involved in a variety of cellular processes, including cell division, adhesion, differentiation, and response to stress. The BCR-ABL1 fusion gene is the hallmark finding in BCR-ABL1-positive chronic myeloid leukemia, but it can also be found in other hematologic neoplasms, including 25-30% of adult B-cell acute lymphoblastic leukemia, 3-5% of pediatric B-cell acute lymphoblastic leukemia, and rarely in acute myeloid leukemia and T-cell acute lymphoblastic leukemia (PMID: 21435002). ABL1 kinase domain mutations, particularly the T315I variant, are associated with resistance to various tyrosine kinase inhibitors in patients with BCR-ABL1-positive leukemias (NCCN Guidelines for Chronic Myelogenous Leukemia).",	
"ABL2":	"ABL2, located on 1q25.2, encodes for a tyrosine kinase involved in a variety of cellular processes, including cell division, adhesion, differentiation, and response to stress. Philadelphia chromosome-like (Ph-like) acute lymphoblastic leukemia is a subgroup of B-cell acute lymphoblastic leukemia associated with an unfavorable prognosis. Although this subgroup is Ph-negative, there is an otherwise similar genetic to the Ph-positive acute lymphoblastic leukemia, with kinase-activating alterations being identified in 91% of patients with Ph-like acute lymphoblastic leukemia. Rearrangements involving ABL2 are one of several kinase-activating lesions in Ph-like acute lymphoblastic leukemia that may be associated with response to tyrosine kinase inhibitors (PMID: 25207766).",	
"ALK":	"ALK, located on 2p23, encodes for a receptor tyrosine kinase belonging to the insulin receptor superfamily. ALK-positive anaplastic large cell lymphoma is characterized by translocations involving the ALK gene and resulting overexpression of the ALK protein. The t(2;5)(p23;q35) involving ALK and NPM1 is the most common type of translocation, accounting for approximately 80% of alterations. ALK-positive anaplastic large cell lymphoma is associated with a more favorable prognosis compared with ALK-negative anaplastic large cell lymphoma. ALK translocations can also be found in a rare variant of diffuse large B-cell lymphoma called ALK-positive large B-cell lymphoma, with the most frequent abnormality in this entity being the t(2;17)(p23;q23) involving ALK and CLTC.",	
"ARHGEF1":	"ARHGEF1, located on 19q13.2, encodes for a protein that forms a complex with G proteins and stimulates Rho-dependent signals. Mutations in the GNA13 effector ARHGEF1 disrupt GNA13 signaling, likely promoting circulation and survival of germinal center B-cells outside the germinal center niche, and are enriched in the germinal center B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 25274307).",	
"ARID1A":	"ARID1A, located on 1p36.11, encodes for a member of the SWI/SNF family, whose members have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. ARID1A mutations are found in 18% of Sezary syndrome (PMID: 26551670), 17% of Waldenstrom macroglobulinemia (PMID: 24366360); 15% of follicular lymphoma, where they are associated with a favorable prognosis (PMID: 26256760); 14% of Burkitt lymphoma (PMID: 23143597), 6% of NK/T-cell lymphoma (PMID: 26192917), and 5% of splenic marginal zone lymphoma (PMID: 22891273).",	
"ARID2":	"ARID2, located on 12q12, encodes for a member of the AT-rich interactive domain (ARID)-containing family of DNA-binding proteins. This protein functions as a subunit of the polybromo and BRG1-associated factor or PBAF (SWI/SNF-B) chromatin remodeling complex, which facilitates ligand-dependent transcriptional activation by nuclear receptors. ARID2 mutations are found 4% of peripheral T-cell lymphoma not otherwise specified (PMID: 25257991).",	
"ASXL1":	"In a patient with AML, mutation in ASXL1 is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, ASXL1 alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival. Somatic mutation in ASXL1 can be found in patients over the age of 50 without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals lack cytopenias and have only a single mutation with low allele frequency. ASXL1 mutations are associated with a poor prognosis in CMML. ASXL1, located on 20q11, encodes for a member of the polycomb family of chromatin-binding proteins, which are involved in epigenetic regulation of gene expression. ASXL1 mutations are commonly found in a variety of myeloid neoplasms, but they appear to be uncommon in lymphoid neoplasms. Somatic mutations in leukemia-associated genes, particularly ASXL1, DNMT3A, or TET2, can also be found in about 10% of elderly individuals without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals, however, lack cytopenias and have only a single mutation with low allele frequency. The presence of a somatic mutation is associated with increases in the risk of developing a hematologic malignancy and in all-cause mortality, with the risk of developing a hematologic malignancy increased by a factor of nearly 50 in individuals with allele frequency of 10% or more (PMID: 25426837).  ASXL1 mutations are associated with a poor prognosis in MDS. ASXL1 mutations are associated with a poor prognosis in MPN. ASXL1 mutations can be seen in various myeloid neoplasms, and are associated with a poor prognosis in MDS, AML, MPN and CMML. ASXL1, located on 20q11, encodes for a member of the polycomb family of chromatin-binding proteins, which are involved in epigenetic regulation of gene expression. ASXL1 mutations are found in 37% of chronic myelomonocytic leukemia (PMID: 27385790), 32% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 29% of refractory anemia with ring sideroblasts associated with marked thrombocytosis (PMID: 26874914), 23% of myelodysplastic syndrome (PMID: 24220272), 22% of primary myelofibrosis (PMID: 23619563), 17% of systemic mastocytosis (PMID: 27214377), 12% of polycythemia vera, 11% of essential thrombocythemia (Tefferi A, et al. Blood Advances 2016;1:21-30), and 11% of acute myeloid leukemia (PMID: 27288520). ASXL1 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, independent of IPSS, IPSS-R, age, and other gene mutations. When mutation status is integrated into the survival analysis by IPSS risk categories, ASXL1 mutations are reported to shift the survival curve of the IPSS risk category to resemble that of the next highest IPSS risk level (PMID: 21714648). Accordingly, current guidelines state that gene mutation analysis can refine the prognosis of myelodysplastic syndrome in patients risk stratified by the IPSS or IPSS-R, and may be helpful in patients predicted to have intermediate risk. ASXL1 mutations are independently associated with unfavorable outcomes and shorter survival after hematopoietic stem cell transplantation in patients with myelodysplastic syndrome and acute myeloid leukemia evolving from myelodysplastic syndrome (PMID: 27601546). ASXL1 mutations are also associated with an unfavorable prognosis in chronic myelomonocytic leukemia (only nonsense and frameshift mutations) (PMID: 27385790), refractory anemia with ring sideroblasts associated with marked thrombocytosis (PMID: 26874914), primary myelofibrosis (PMID: 23619563), polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30), systemic mastocytosis (PMID: 27214377), and acute myeloid leukemia with wild-type FLT3-ITD and normal karyotype or intermediate-risk cytogenetic abnormalities (PMID: 22417203). ASXL1 mutations are reported to be highly specific for secondary acute myeloid leukemia, and may also be helpful in identifying a subset of elderly patients with de novo acute myeloid leukemia with worse clinical outcomes (PMID: 25550361). Somatic mutations in leukemia-associated genes, particularly ASXL1, DNMT3A, or TET2, can also be found in about 10% of elderly individuals without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals, however, lack cytopenias and have only a single mutation with low allele frequency. The presence of a somatic mutation is associated with increases in the risk of developing a hematologic malignancy and in all-cause mortality, with the risk of developing a hematologic malignancy increased by a factor of nearly 50 in individuals with allele frequency of 10% or more (PMID: 25426837). Somatic mutations in leukemia-associated genes are also commonly found in patients with unexplained cytopenias who do not meet diagnostic criteria for myelodysplastic syndrome, a condition known as clonal cytopenias of undetermined significance (CCUS) (PMID: 26429975 and PMID: 26392596).",	
"ATM":	"ATM, located on 11q22-q23, encodes for a cell cycle checkpoint kinase belonging to the PI3/PI4-kinase family. ATM mutations are found in 12% of chronic lymphocytic leukemia, and are associated with unmutated IGHV, 11q deletion, and an unfavorable prognosis, particularly when 11q deletion is also present (PMID: 17968022 and PMID: 23091097). ATM mutations are also in 40-75% of mantle cell lymphoma (PMID: 22210878 and PMID: 24145436), 46% of T-cell prolymphocytic leukemia (PMID: 9288106); and 4% of multiple myeloma, where they are associated with an unfavorable prognosis (PMID: 26282654).",	
"ATRX":	"Somatic mutations in ATRX can be seen in a rare subtype of MDS associated with alpha thalassemia.",	
"B2M":	"B2M, located on 15q21.1, encodes for a serum protein found in association with the major histocompatibility complex (MHC) class I heavy chain on the surface of nearly all nucleated cells. In 29% of diffuse large B-cell lymphoma, mutations or deletions inactivate B2M, thus preventing the cell-surface expression of MHC class-I complex that is necessary for recognition by CD8-positive cytotoxic T-cells (PMID: 22137). B2M mutations are also found in 4% of adult T-cell leukemia/lymphoma (PMID: 26437031).",	

"BCL2":	"BCL2, located on 18q21.3, encodes for an integral outer mitochondrial membrane protein that blocks the apoptotic death of certain cells such as lymphocytes. Overexpression of BCL2 due to translocation of BCL2 and IGH is a hallmark finding in follicular lymphoma, where it is found in approximately 90% of cases. Overexpression of BCL2 through this translocation can also be found in 30-40% of the germinal center B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 25499448). Concurrent BCL2 and MYC rearrangement defines a subset of high-grade B-cell lymphomas known as double-hit lymphomas, which are associated with an unfavorable prognosis and poor response to conventional therapy. Translocation of BCL2 and MALT1 are found in 10-15% of extranodal marginal zone lymphoma of mucosa-associated lymphoid tissue. Aside from translocations, mutations in BCL2 have also been observed in lymphomas, and are found in 18-23% of diffuse large B-cell lymphoma, often restricted to the germinal center B-cell-like subtype (PMID: 23699601 and PMID: 23292937); 12% of follicular lymphoma at diagnosis but 53% at transformation, where they are associated with an unfavorable prognosis (PMID: 25452615); and 7% of Burkitt lymphoma (PMID: 23143597).",	
"BCL6":	"BCL6, located on 3q27, encodes for a zinc finger transcription factor that acts as a sequence-specific repressor of transcription, modulating the transcription of STAT-dependent IL-4 responses of B-cells. BCL6 rearrangements are found in 48% of nodular lymphocyte predominance Hodgkin lymphoma, but not classical Hodgkin lymphoma (PMID: 12393409); 19% of diffuse large B-cell lymphoma, with higher frequencies in the primary mediastinal (33%) and activated B-cell-like (24%) subtypes than the germinal center B-cell-like subtype (PMID: 17625604); 19% of t(14;18)-positive follicular lymphoma but 26% of t(14;18)-negative and 60% of grade 3 t(14;18)-negative follicular lymphoma (PMID: 19465899); and a subset of double/triple-hit lymphoma. Aside from BCL6 rearrangements, mutations in BCL6 have also been observed in lymphomas, and are found in 15% of diffuse large B-cell lymphoma (PMID: 23699601), 7% of Burkitt lymphoma (PMID: 23143597), and 5% of follicular lymphoma (PMID: 25713363).",	
"BCOR":	"In a patient with AML, mutation in BCOR is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, BCOR alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival.BCOR, located on Xp11.4, encodes for a transcriptional co-repressor of BCL6 that is required for germinal center formation and may influence apoptosis. BCOR mutations are found in 21% of extranodal NK/T-cell lymphoma nasal type (PMID: 25980440), 8% of T-cell prolymphocytic leukemia (PMID: 26493028), 7% of splenic marginal zone lymphoma (PMID: 24296945), and 2% of chronic lymphocytic leukemia (PMID: 26466571). BCOR mutations can be seen in various myeloid neoplasms, and are associated with a poor prognosis in MDS, MPN, and AML.",	
"BIRC3":	"BIRC3, located on 11q22, encodes for a member of the IAP family of proteins, which inhibit apoptosis by binding to tumor necrosis factor receptor-associated factors TRAF1 and TRAF2. BIRC3 mutations or deletions lead to activation of the alternative NF-kappaB pathway, are found in 4% of chronic lymphocytic leukemia at diagnosis but 24% of fludarabine refractory chronic lymphocytic leukemia, and are associated with 11q deletion and an unfavorable prognosis (PMID: 22308293 and PMID: 23243274). BIRC3 alterations are also found in a subset of splenic marginal zone lymphoma (5% mutations, 6% deletions) (PMID: 22891273); and 6% of mantle cell lymphoma (PMID: 24145436), where they are reportedly associated with resistance to the BTK inhibitor ibrutinib (PMID: 24362935).",	
"BRAF":	"BRAF, located on 7p34, is part of the MAP kinase pathway, and mutations in BRAF lead to constitutive activation of downstream signaling. BRAF V600 mutations are found in a wide variety of neoplasms. Among hematopoietic neoplasms, BRAF V600 mutations are found in virtually all classic forms of hairy cell leukemia (PMID: 21663470), 57% of Langerhans cell histiocytosis (PMID: 20519626), and 54% Erdheim-Chester disease (PMID: 22879539); where they are reportedly associated with an increased likelihood of response to BRAF inhibitor (PMID: 26352686 and PMID: 26287849). BRAF mutations are also found in 7% of multiple myeloma (PMID: 26282654) and 4% chronic lymphocytic leukemia (PMID: 26466571). In chronic lymphocytic leukemia, however, the BRAF mutation typically does not involve the canonical hotspot V600 seen in other neoplasms. BRAF, located on 7q34, encodes for a protein that regulates the MAP kinase/ERK signaling pathway, affecting cell division, differentiation, and secretion. BRAF mutations lead to constitutive activation of downstream signaling, and are found in a wide variety of neoplasms, including virtually all classic forms of hairy cell leukemia; approximately half of melanomas, thyroid cancers, Langerhans cell histiocytosis, and Erdheim-Chester disease; and a smaller subset of colorectal cancers, non-small cell lung cancers, ovarian cancers, and gliomas. BRAF mutations are rarely found in myeloid neoplasms (PMID: 24030381).",
"BTK":	"BTK, located on Xq21.33-q22, encodes for a tyrosine kinase important in B-cell development, differentiation, and signaling. Germline BTK mutations lead to X-linked agammaglobulinemia. The BTK inhibitor ibrutinib irreversibly binds BTK at the C481 residue, rendering it kinase-inactive, inducing apoptosis, and abolishing proliferation and B-cell receptor signaling in vitro. The BTK C481S mutation reportedly confers resistance to ibrutinib by preventing this irreversible binding (PMID: 24869598).",	
"CALR":	"CALR mutations are included as one of the major diagnostic criteria for ET in the WHO classification. CALR mutations are associated with lower risk of thrombosis in ET compared to JAK2 mutations. CALR mutations are frequently seen in MPN and occasionally seen in MDS/MPN. CALR mutations are included as one of the major diagnostic criteria for ET and PMF in the WHO classification. CALR, located on 19p13.13, encodes for a multifunctional protein that acts as a major Ca(2+)-binding protein in the lumen of the endoplasmic reticulum. It is also found in the nucleus, suggesting that it may have a role in transcription regulation. CALR mutations are found in 60-90% of primary myelofibrosis and 50-70% of essential thrombocythemia that are negative for JAK2 and MPL mutations (PMID: 24325359, PMID: 24325356), and are included as one of the major diagnostic criteria for primary myelofibrosis and essential thrombocythemia in the 2016 revision of the WHO classification (PMID: 27069254). CALR mutations are associated with a more favorable prognosis and lower risk of thrombosis in patients with primary myelofibrosis and essential thrombocythemia compared to JAK2 mutations (PMID: 24325356, PMID: 24496303, PMID: 25482134). The favorable prognostic effect of CALR mutations in primary myelofibrosis, however, appears restricted to type 1 (52-bp deletion, p.L367fs*46) or type 1-like variants (PMID: 25301336).CALR mutations are included as one of the major diagnostic criteria for PMF in the WHO classification. CALR mutations are associated with a more favorable prognosis compared to JAK2 mutation and triple-negative PMF, and this favorable prognostic effect appears restricted to type 1/type 1-like CALR mutations.CALR mutations are frequently seen in MPN, while SF3B1 mutations are frequently seen in MDS with ring sideroblasts. Co-mutation of CALR and SF3B1 is suggestive of MDS/MPN-RS-T.",	
"CARD11":	"CARD11, located on 7p22, encodes for a protein belonging to the membrane-associated guanylate kinase (MAGUK) family of proteins, which function as molecular scaffolds for the assembly of multiprotein complexes at specialized regions of the plasma membrane. CARD11 mutations are found in 30% of primary central nervous system lymphoma (PMID: 25991819), 24% of adult T-cell leukemia/lymphoma (PMID: 26437031), 15% of Sezary syndrome (PMID: 26551670); 12% of follicular lymphoma, where they are associated with a more unfavorable prognosis (PMID: 26256760); 10% of the activated B-cell-like subtype versus 4% of the germinal center B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 18323416); 7% of splenic marginal zone lymphoma (PMID: 22891273), and 4% of angioimmunoblastic and other follicular helper T-cell-derived lymphomas (PMID: 27369867), and 1% of chronic lymphocytic leukemia (PMID: 26466571).",	


"CBL":	"Germline CBL mutations are included as one of the genetic criteria for JMML in the WHO classification. Most children with JMML and germline CBL mutations experience spontaneous regression of JMML. CBL mutations are associated with a poor prognosis in MPN. CBL mutations can be seen in various myeloid neoplasms, more frequently in CMML and JMML. CBL mutations are associated with a poor prognosis in MPN. CBL, located on 11q23.3, encodes for a RING finger E3 ubiquitin ligase that functions as a negative regulator of many signal transduction pathways. CBL mutations are found in 17% of juvenile myelomonocytic leukemia (PMID: 19571318), 11% of systemic mastocytosis (PMID: 27214377), 8% of chronic myelomonocytic leukemia (PMID: 27385790), 8% of atypical chronic myeloid leukemia (PMID: 19387008), 5% of myelodysplastic syndrome (PMID: 24220272), 4% of primary myelofibrosis (PMID: 23619563), and 2% of acute myeloid leukemia (PMID: 27288520).",	
"CCND1":	"CCND1, located on 11q13, encodes for a protein required for cell cycle G1/S transition. CCND1-IGH translocation is found in >95% of mantle cell lymphoma (PMID: 24145436), 59% of AL amyloidosis (PMID: 27257181), and 15% of multiple myeloma (PMID: 26282654), while CCND1 mutations are found in 35% of mantle cell lymphoma (PMID: 24145436) and 2% of multiple myeloma (PMID: 26282654). In multiple myeloma, CCND1-IGH translocation is associated with CD20 expression, lymphoplasmacytic morphology, hyposecretory disease, lambda light chain restriction, and neutral effect on prognosis (PMID: 19798094), while CCND1 mutations are associated with an unfavorable prognosis (PMID: 26282654). In AL amyloidosis, CCDN1-IGH translocation is associated with a favorable prognosis in patients treated with high-dose melphalan (PMID: 27257181).",	
"CCND2":	"CCND2, located on 12p13.32, encodes for a protein required for cell cycle G1/S transition. CCND2 translocations, predominantly involving immunoglobulin light chain genes, are found in 55% of cyclin D1-negative mantle cell lymphoma (PMID: 23255553).",	
"CCND3":	"CCND3, located on 6p21.1, encodes for a protein required for cell cycle G1/S transition. CCND3 mutations are found in 67% of HIV-associated Burkitt lymphoma, 38% of sporadic Burkitt lymphoma, but only 2% of endemic Burkitt lymphoma, indicating a distinct genetic pathogenesis for the latter Burkitt lymphoma subtype (PMID: 22885699). CCND3 mutations are also found in 24% of splenic diffuse red pulp lymphoma (PMID: 28069605), 11% of the activated B-cell-like subtype of diffuse large B-cell lymphoma, and 2% of the germinal center B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 22885699).",	
"CD274":	"CD274 (also known as PDL1), located on 9p24, encodes for an immune inhibitory receptor ligand that is expressed by hematopoietic and non-hematopoietic cells and various types of tumor cells. In tumor microenvironments, interaction of PDL1 with its receptor PD1 inhibits cytotoxic T-cell activation and cytokine production, providing an immune escape for tumor cells. Alterations of the PDL locus on 9q24.1, resulting in overexpression of PDL, are found in primary mediastinal B-cell lymphoma (63% amplification, 20% translocation), classical Hodgkin lymphoma (38% amplification), and diffuse large B-cell lymphoma (14% gain, 3% amplification) (PMID: 20628145 and PMID: 24497532). Overexpression of PDL1 as measured by PDL1 and PAX5 double immunohistochemical staining is found in 11% of diffuse large B-cell lymphoma, and is associated with non-germinal center B-cell-like subtype, Epstein-Barr virus positivity, and an unfavorable prognosis (PMID: 26239088).",	
"CD79A":	"The B-cell antigen receptor is a multimeric complex that includes the antigen-specific component, surface immunoglobulin (Ig). Surface Ig non-covalently associates with two other proteins, Ig-alpha and Ig-beta, which are necessary for expression and function of the B-cell antigen receptor. CD79A, located on 19q13.2, encodes for the Ig-alpha protein of the B-cell antigen component. CD79A mutations are found in 9% of primary central nervous system lymphoma (PMID: 25643939), 6% of Waldenstrom macroglobulinemia (PMID: 23861223), and 3% of the activated B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 20054396).",	
"CD79B":	"The B-cell antigen receptor is a multimeric complex that includes the antigen-specific component, surface immunoglobulin (Ig). Surface Ig non-covalently associates with two other proteins, Ig-alpha and Ig-beta, which are necessary for expression and function of the B-cell antigen receptor. CD79B, located on 17q23, encodes for the Ig-beta protein of the B-cell antigen component. CD79B mutations are found in 18% of the activated B-cell-like subtype of diffuse large B-cell lymphoma, and are thought to result in chronic active BCR signaling and constitutive activation of the NF-kappaB pathway (PMID: 20054396). CD79B mutations are also found in 40% of primary central nervous system lymphoma (PMID: 25991819), 7% of Waldenstrom macroglobulinemia (PMID: 24366360), and 5% of follicular lymphoma (PMID: 28064239).",	
"CDKN1B":	"CDKN1B, located on 12p13.1, encodes for a cyclin-dependent kinase inhibitor that binds to and prevents the activation of cyclin E-CDK2 or cyclin D-CDK4 complexes, thus, controlling the cell cycle progression at G1. Deletions of 12p13.1 with loss of CDKN1B are found in 20% of Sezary syndrome (PMID: 26551667), 12% of pediatric T-cell acute lymphoblastic leukemia (PMID: 19406988); and 12% of mantle cell lymphoma, where they are associated with an unfavorable prognosis independent of MIPI (PMID: 26022239).",	
"CDKN2A":	"CDKN2A, located on 9p21, is a tumor suppressor gene that is frequently deleted in a wide variety of neoplasms. Deletions of CDKN2A or the adjacent CDKN2B are associated with lymphoma transformation, and are found in 71% of transformed mycosis fungoides (PMID: 20118908), 46% of transformed follicular lymphoma (PMID: 24388756), and 20% of Richter syndrome (PMID: 24004666). Deletions of CDKN2A/B are also found in 60% of primary central nervous system lymphoma (PMID: 25991819), 50% of pediatric T-cell and 21% of B-cell acute lymphoblastic leukemia (PMID: 18838613); 25% of mantle cell lymphoma, where they are associated with an unfavorable prognosis independent of MIPI (PMID: 26022239); and 9% of nodal marginal zone lymphoma (PMID: 27335277).",	
"CDKN2B":	"CDKN2B, located on 9p21.3, lies adjacent to the tumor suppressor gene CDKN2A in a region that is frequently deleted in a wide variety of neoplasms. Deletions of CDKN2B or the adjacent CDKN2A are associated with lymphoma transformation, and are found in 71% of transformed mycosis fungoides (PMID: 20118908), 46% of transformed follicular lymphoma (PMID: 24388756), and 20% of Richter syndrome (PMID: 24004666). Deletions of CDKN2A/B are also found in 60% of primary central nervous system lymphoma (PMID: 25991819), 50% of pediatric T-cell and 21% of B-cell acute lymphoblastic leukemia (PMID: 18838613); 25% of mantle cell lymphoma, where they are associated with an unfavorable prognosis independent of MIPI (PMID: 26022239); and 9% of nodal marginal zone lymphoma (PMID: 27335277).",	
"CEBPA":	"A monoallelic (single) CEBPA mutation is detected. Cytogenetically normal AML patients carrying a monoallelic (single) CEBPA mutation show no difference in outcome compared to patients with non-mutated (wildtype) CEBPA. Biallelic (double) CEBPA mutations are detected. Cytogenetically normal AML patients with biallelic (double) CEBPA gene mutations represent a molecularly distinct group with a favorable prognosis. CEBPA, located on 19q13.1, encodes for a transcription factor that modulates the expression of genes involved in cell cycle regulation and body weight homeostasis. CEBPA mutations are found in 8% of acute myeloid leukemia (PMID: 27288520), and are associated with normal karyotype and favorable risk status in the absence of FLT3-ITD mutations and other adverse prognostic factors. The favorable prognostic effect of CEBPA mutations, however, appears restricted to patients with biallelic (double) CEBPA mutations (PMID: 20439648). Acute myeloid leukemia with biallelic (double) mutations of CEBPA is categorized as an acute myeloid leukemia with recurrent genetic abnormality in the 2016 revision of the WHO classification. Acute myeloid leukemia that occurs on the background of a predisposing germline CEBPA mutation is also included in the 2016 revision of the WHO classification , and is categorized as an acute myeloid leukemia with germline CEBPA mutation (PMID: 27069254).",	
"CIITA":	"CIITA, located on 16p13, encodes for a protein that acts as a positive regulator of class II major histocompatibility (MHC) complex gene transcription. Disruption of CIITA leads to decreased surface expression of MHC II and reduced tumor cell immunogenicity. Translocations involving CIITA are found in 38% of primary mediastinal B-cell lymphoma and 15% of classical Hodgkin lymphoma (PMID: 21368758), while CIITA mutations are found in 10% of diffuse large B-cell lymphoma (PMID: 22343534).",		
"CREBBP":	"CREBBP, located on 16p13.3, encodes for a histone acetyltransferase that also has a role in acetylation of BCL6 and TP53. CREBBP mutations are found in 70% of follicular lymphoma, where they are associated with a more unfavorable prognosis (PMID: 26256760); 22% of diffuse large B-cell lymphoma, with enrichment in the germinal center B-cell-like subtype; and 12% of Burkitt lymphoma (PMID: 23143597). CREBBP mutations are also found in 18% of relapsed B-cell acute lymphoblastic leukemia, where they are associated with resistance to glucocorticoids (PMID: 21390130).",	
"CRLF2":	"CRLF2, located on Xp22.33 and Yp11.2, encodes for the receptor for thymic stromal lymphopoietin (TSLP). Together with the interleukin 7 receptor, CRLF2 and TSLP activate the STAT3, STAT5, and JAK2 pathways, which control processes such as cell proliferation and development of the hematopoietic system. Rearrangements involving CRLF2 are found in 47% of Philadelphia chromosome (Ph)-like acute lymphoblastic leukemia, with concomitant JAK1 or JAK2 mutation found in 55% of such cases. Ph-like acute lymphoblastic leukemia may be associated with response to tyrosine kinase inhibitors (PMID: 25207766). CRLF2 mutations in myeloid neoplasms are rare, and their prognostic significance has not been defined.",	
"CSF1R":	"CSF1R, located on 5q32, encodes for a tyrosine kinase transmembrane receptor and member of the CSF1/PDGF receptor family of tyrosine-protein kinases. Philadelphia chromosome-like (Ph-like) acute lymphoblastic leukemia is a subgroup of B-cell acute lymphoblastic leukemia associated with an unfavorable prognosis. Although this subgroup is Ph-negative, there is an otherwise similar genetic to the Ph-positive acute lymphoblastic leukemia, with kinase-activating alterations being identified in 91% of patients with Ph-like acute lymphoblastic leukemia. Rearrangements involving CSF1R are one of several kinase-activating lesions in Ph-like acute lymphoblastic leukemia that may be associated with response to tyrosine kinase inhibitors (PMID: 25207766).",	


"CSF3R":	"CSF3R T618I and other activating CSF3R mutations are included as one of the diagnostic criteria for CNL in the WHO classification. CSF3R, located on 1p35-p34.3, encodes for a receptor for colony-stimulating factor 3, a cytokine that controls production, differentiation, and function of granulocytes. CSF3R mutations are found in 83% of chronic neutrophilic leukemia (PMID: 23604229), and are included as one of the diagnostic criteria for chronic neutrophilic leukemia in the 2016 revision of the WHO classification (PMID: 27069254). Also, while CSF3R mutations are overall rare in acute myeloid leukemia, they appear enriched in acute myeloid leukemia with biallelic CEBPA mutations (PMID: 27034432, PMID: 27143256). CSF3R mutations may be associated with sensitivity to JAK inhibitors (PMID: 23656643, PMID: 27034432).",	
"CTCF":	"CTCF, located on 16p21-q22.3, is a member of the BORIS + CTCF gene family and it encodes a transcriptional regulator protein that can either bind a histone acetyltransferase-containing complex and function as a transcriptional activator or bind a histone deacetylase-containing complex and function as a transcriptional repressor. CTCF mutations are uncommon in lymphoid neoplasms.",	
"CTNNB1":	"Limited studies have shown mutations in CTNNB1 to be associated with shorter overall survival in cytogenetically normal AML. CTNNB1, located on 3p22.1, encodes for a protein called beta-catenin, which play an important role in cell adhesion and communication between cells. It is also involved in cell signaling as an essential part of the WNT signaling pathway. CTNNB1 mutations are found in 19% of nasal NK/T-cell lymphoma (PMID: 14745729) and 6% of angioimmunoblastic and other follicular helper T-cell-derived lymphomas (PMID: 27369867).",	
"CUX1":	"CUX1 mutations are associated with a poor prognosis in MPN. Deletion of 7q with loss of the CUX1 locus is associated with an adverse prognosis in myeloid neoplasms; however, the significance of CUX1 loss in the absence of 7q deletion is uncertain.",
"CXCR4":	"CXCR4, located on 2q21, encodes for a protein that plays a crucial role in modulating cell trafficking in hematopoietic stem cells and clonal B-cells. CXCR4 mutations can be found in an inherited heterozygous autosomal dominant immune deficiency disorder called WHIM (warts, hypogammaglobulinemia, infections, and myelokathexis) syndrome. WHIM-like somatic mutations are found in 28% of Waldenstrom macroglobulinemia, 20% of IgM monoclonal gammopathy of undetermined significance, 7% of splenic marginal zone lymphoma, 4% of follicular lymphoma (PMID: 28064239), and 1% of diffuse large B-cell lymphoma (PMID: 24711662). The mutation status of CXCR4 and MYD88 in Waldenstrom macroglobulinemia may help predict responsiveness to the BTK inhibitor ibrutinib, with response rates reportedly highest among patients with wild-type CXCR4 and mutated MYD88, followed by patients with mutated CXCR4 and MYD88, and patients with wild-type CXCR4 and MYD88 (PMID: 25853747).",
	
"DDX3X":	"DDX3X, located on DDX3X, encodes for an ATP-dependent RNA helicase involved in several steps of RNA processing pathways, including transcription, translation initiation, splicing, and mRNA export. DDX3X mutations are found in 32% of Burkitt lymphoma (PMID: 22885699), 20% of NK/T-cell lymphoma (PMID: 26192917), 5% of chronic lymphocytic leukemia (PMID: 27226433).",
"DDX41":	"DDX41, located on 5q35.3, encodes for a member of the DEAD box family of proteins, which are implicated in a number of cellular processes involving alteration of RNA secondary structure, such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Familial myelodysplastic syndrome/acute myeloid leukemia with germline DDX41 mutations are characterized by long latency, advanced disease, normal karyotype, and unfavorable prognosis. Germline DDX41 mutations can also be found in sporadic cases of myeloid neoplasms, where they likely serve as a genetic first hit and predispose to further somatic DDX41 mutations as a secondary hit (PMID: 25920683). Myeloid neoplasms that occur on the background of a predisposing germline DDX41 mutation are categorized as myeloid neoplasms with germline DDX41 mutation in the 2016 revision of the WHO classification (PMID: 27069254). DDX41 mutations are associated with an increased response to lenalidomide in myeloid neoplasms (PMID: 25920683, PMID: 27560106).",	
"DIS3":	"DIS3, located on 13q21.33, encodes for a component of the RNA exosome complex, which is involved in RNA processing and degradation. DIS3 mutations are found in 9% of multiple myeloma, with enrichment in cases with t(4;14) (25%) and t(11;14) (12%) (PMID: 26282654).",	
"DNMT3A":	"Somatic mutation in DNMT3A can be found in patients over the age of 50 without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals lack cytopenias and have only a single mutation with low allele frequency. DNMT3A, located on 2p23, encodes for a protein involved in epigenetic regulation critical for the silencing of tumor suppressor genes associated with hematopoietic neoplasms. DNMT3A mutations are commonly found in a variety of myeloid neoplasms. Among lymphoid neoplasms, DNMT3A mutations are found in 27% of peripheral T-cell lymphoma not otherwise specified, 26% of angioimmunoblastic T-cell lymphoma (PMID: 24413737); 16% of adult early T-cell precursor acute lymphoblastic leukemia (PMID: 23603912), where they are associated with a unfavorable prognosis (PMID: 23603912 and PMID: 23687089); and 7% of hepatosplenic T-cell lymphoma (PMID: 28122867).Somatic mutations in leukemia-associated genes, particularly ASXL1, DNMT3A, or TET2, can also be found in about 10% of elderly individuals without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals, however, lack cytopenias and have only a single mutation with low allele frequency. The presence of a somatic mutation is associated with increases in the risk of developing a hematologic malignancy and in all-cause mortality, with the risk of developing a hematologic malignancy increased by a factor of nearly 50 in individuals with allele frequency of 10% or more (PMID: 25426837). Somatic mutations in leukemia-associated genes are also commonly found in patients with unexplained cytopenias who do not meet diagnostic criteria for myelodysplastic syndrome, a condition known as clonal cytopenias of undetermined significance (CCUS) (PMID: 26429975 and PMID: 26392596).  DNMT3A mutations are associated with a poor prognosis in MDS after hematopoietic stem cell transplantation. DNMT3A mutations are associated with worse overall survival in patients with myelofibrosis undergoing allogenic hematopoietic stem cell transplantation. DNMT3A mutations can be seen in various myeloid neoplasms and are associated with a poor prognosis in MDS after hematopoietic stem cell transplantation. DNMT3A, located on 2p23, encodes for a protein involved in epigenetic regulation critical for the silencing of tumor suppressor genes associated with hematopoietic neoplasms. DNMT3A mutations are found in 31% of acute myeloid leukemia (PMID: 27288520), 13% of myelodysplastic syndrome (PMID: 24220272), 13% of refractory anemia with ring sideroblasts associated with marked thrombocytosis (PMID: 26874914), 8% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 6% of primary myelofibrosis (PMID: 23619563), 6% of essential thrombocythemia, 2% of polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30), and 4% of chronic myelomonocytic leukemia (PMID: 27385790). DNMT3A mutations are associated with an unfavorable prognosis in myelodysplastic syndrome when SF3B1 is not mutated, and also unfavorable outcomes and shorter survival after hematopoietic stem cell transplantation (PMID: 25092778). DNMT3A mutations are associated with an unfavorable prognosis in acute myeloid leukemia with FLT3-ITD mutation and normal karyotype or intermediate-risk cytogenetic abnormalities, but improved outcome in patients treated with high-dose induction chemotherapy compared with standard-dose induction chemotherapy (PMID: 22417203).",	
"EBF1":	"EBF1, located on 5q33.3, encodes for a transcription factor that is essential for specification in early B-cell development. Philadelphia chromosome-like (Ph-like) acute lymphoblastic leukemia is a subgroup of B-cell acute lymphoblastic leukemia associated with an unfavorable prognosis. Although this subgroup is Ph-negative, there is an otherwise similar genetic to the Ph-positive acute lymphoblastic leukemia, with kinase-activating alterations being identified in 91% of patients with Ph-like acute lymphoblastic leukemia. EBF1-PDGFRB fusion is one of several kinase-activating lesions in Ph-like acute lymphoblastic leukemia that may be associated with response to tyrosine kinase inhibitors (PMID: 25207766).",	
"EGR1":	"EGR1, located on 5q31.2, encodes for a member of the EGR family of C2H2-type zinc-finger proteins. This transcriptional regulator cross-regulates and interacts with the CREEP/EP300 complex, is induced by BCR signaling, and is essential to marginal B-cell development. EGR1 mutations are found at a low frequency in lymphoid neoplasms, including 5% of follicular lymphoma (PMID: 28064239), 5% of hyperdiploid multiple myeloma (PMID: 26282654), and 1% of splenic marginal zone lymphoma (PMID: 22891273).",

	
"EP300":	"EP300, located on 22q13.2, encodes for a protein that functions as a histone acetyltransferase, regulating transcription via chromatin remodeling. EP300 mutations are found in 14% of follicular lymphoma, where they are associated with an unfavorable prognosis (PMID: 26256760); 10% of diffuse large B-cell lymphoma, with enrichment in the germinal center B-cell-like subtype (PMID: 21390126); 6% of adult T-cell leukemia/lymphoma (PMID: 26437031), 5% of splenic marginal zone lymphoma (PMID: 22891273), and 4% of NK/T-cell lymphoma (PMID: 26192917).",	
"EPOR":	"EPOR, located on 19p13.2, encodes for the erythropoietin receptor, which is a member of the cytokine receptor family. Upon erythropoietin binding, this receptor activates JAK2 tyrosine kinase which in turn activates different intracellular pathways including RAS/MAP kinase, phosphatidylinositol 3-kinase, and STAT transcription factors. Philadelphia chromosome-like (Ph-like) acute lymphoblastic leukemia is a subgroup of B-cell acute lymphoblastic leukemia associated with an unfavorable prognosis. Although this subgroup is Ph-negative, there is an otherwise similar genetic to the Ph-positive acute lymphoblastic leukemia, with kinase-activating alterations being identified in 91% of patients with Ph-like acute lymphoblastic leukemia. Rearrangements involving EPOR are one of several kinase-activating lesions in Ph-like acute lymphoblastic leukemia that may be associated with response to tyrosine kinase inhibitors (PMID: 25207766).",	
"ETNK1":	"ETNK1, located on 12p12.1, encodes for an ethanolamine kinase, which functions in the first committed step of the phosphatidylethanolamine synthesis pathway. ETNK1 mutations are found in 20% of systemic mastocytosis with associated eosinophilia (PMID: 25615281), 9% of atypical chronic myeloid leukemia (PMID: 25343957), and 3% of chronic myelomonocytic leukemia (PMID: 27385790). The presence of an ETNK1 mutation provides supportive evidence for the diagnosis of atypical chronic myeloid leukemia in the 2016 revision of the WHO classification (PMID: 27069254).  ETNK1 mutations can be seen in various myeloid neoplasms, including atypical BCR-ABL1-negative CML, where they can provide supportive evidence for this diagnosis per the WHO classification.",	

"ETV6":	"Myeloid neoplasms that occur on the background of a predisposing germline ETV6 mutation are categorized as myeloid neoplasms with germline predisposition in the WHO classification. ETV6, located on 12p13, encodes for an ETS family transcription factor that contains two functional domains: a N-terminal pointed (PNT) domain that is involved in protein-protein interactions with itself and other proteins, and a C-terminal DNA-binding domain. The t(12;21)(p13;q22), resulting in the ETV6-RUNX1 gene fusion, is found in 22% of pediatric and 2% of adult B-cell acute lymphoblastic leukemia, and is associated with a favorable prognosis. ETV6 mutations are Independently associated with a poor prognosis in MDS.  ETV6, located on 12p13, encodes for a member of the ETS family of transcription factors, which are necessary for hematopoiesis and maintenance of the developing vascular network. ETV6 mutations are found in 12% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 2% of myelodysplastic syndrome (PMID: 24220272), and 2% of acute myeloid leukemia (PMID: 27288520). Myeloid neoplasms that occur on the background of a predisposing germline ETV6 mutation are categorized as myeloid neoplasms with germline ETV6 mutation in the 2016 revision of the WHO classification (PMID: 27069254). ETV6 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, independent of IPSS, IPSS-R, age, and other gene mutations. When mutation status is integrated into the survival analysis by IPSS risk categories, ETV6 mutations are reported to shift the survival curve of the IPSS risk category to resemble that of the next highest IPSS risk level (PMID: 21714648). Accordingly, current guidelines state that gene mutation analysis can refine the prognosis of myelodysplastic syndrome in patients risk stratified by the IPSS or IPSS-R, and may be helpful in patients predicted to have intermediate risk.",
"EZH2":	"In a patient with AML, mutation in EZH2 is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, EZH2 alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival. EZH2 mutations are associated with a poor prognosis in ET. EZH2, located on 7q35-q36, encodes for a member of the Polycomb-group (PcG) family of proteins, which are which are involved in maintaining the transcriptional repressive state of genes over successive cell generations. EZH2 mutations are found in 22% of follicular lymphoma, where they are associated with a favorable prognosis (PMID: 26256760); 22% of the germinal center B-cell-like subtype of diffuse large B-cell lymphoma but not the activated B-cell-like subtype (PMID: 20081860), 10% of T-cell prolymphocytic leukemia (PMID: 24825865), and 7% of Burkitt lymphoma (PMID: 23143597). Follicular lymphoma with EZH2 mutation may respond to EZH2 inhibitors. EZH2 mutations are associated with a poor prognosis in MDS. EZH2 are associated with a poor prognosis in MPN. EZH2 mutations can be seen in various myeloid neoplasms and are associated with a poor prognosis in MDS and MPN. EZH2 mutations are associated with a poor prognosis in PMF.",	
"FAM46C":	"FAM46C, located on 1p12, encodes for a protein that may be functionally involved with type 1 interferon response. FAM46C mutations are found in 6% of multiple myeloma (PMID: 26282654).",	
"FAS":	"FAS, located on 10q23.31, encodes for a receptor that plays a central role in the physiological regulation of programmed cell death, and has been implicated in the pathogenesis of various neoplasms and diseases of the immune system. This receptor has also been shown to activate NF-kappaB, MAPK3/ERK1, and MAPK8/JNK. Germline FAS mutations are found in the majority of autoimmune lymphoproliferative disorder, while somatic FAS mutations are found in 39% of genetically undefined autoimmune lymphoproliferative disorder (PMID: 20360470). FAS mutations are also found in 11% of sporadic non-Hodgkin lymphoma, where they are associated with extranodal disease and clinical features suggestive of autoimmune disease (PMID: 9787134).",	
"FAT1":	"FAT1, located on 4q35, encodes for a tumor suppressor protein essential for controlling cell proliferation during development. FAT1 mutations have been shown to negatively regulate WNT signaling, and are found in 10% of fludarabine refractory chronic lymphocytic leukemia but only 1% of unselected chronic lymphocytic leukemia at diagnosis, suggesting a role in the development of a high-risk phenotype (PMID: 24550227). FAT1 mutations are also found in 25% of adult early T-cell precursor acute lymphoblastic leukemia (PMID: 23603912), 12% of adult T-cell acute lymphoblastic leukemia (PMID: 24972153), and 7% of splenic marginal zone lymphoma (PMID: 24296945).",	
"FBXW7":	"FBXW7, located on 4q31.3, encodes for a member of the F-box family of proteins, which are substrate recognition components of a multi-subunit ubiquitin ligase that mediate the ubiquitin-dependent proteolysis of several oncoproteins. FBXW7 has been shown to be mutated in a wide variety of neoplasms, and it is thought that FBXW7 likely functions as a tumor suppressor. FBXW7 mutations are found in 24% of adult T-cell acute lymphoblastic leukemia (PMID: 19109228), where they are associated with a favorable prognosis in the absence of RAS or PTEN abnormalities (PMID: 24166518); 18% of pediatric T-cell acute lymphoblastic leukemia (PMID: 22814294), and 6% of chronic lymphocytic leukemia (PMID: 27226433).",	
"FGFR3":	"FGFR3, located on 4p16.3, encodes for a member of the fibroblast growth factor receptor (FGFR) family, which plays a role in several important cellular processes, including proliferation, differentiation, angiogenesis, wound healing, and embryo development. The t(4;14)(p23;q35) involving FGFR3 and IGH is found in 13% of multiple myeloma, and is associated with an unfavorable prognosis. Activating FGFR3 mutations appear to be restricted in the t(4;14) subgroup of multiple myeloma, where they are found in 17% of such cases (PMID: 26282654). FGFR3 mutations are also found in 5% of Burkitt lymphoma (PMID: 23143597).",	
"FLT3":	"The FLT3-ITD allelic ratio is high (>0.5). FLT3-ITD with high allelic ratio and mutated NPM1 is associated with an intermediate prognosis in AML. The FLT3-ITD allelic ratio is high (>0.5). FLT3-ITD with high allelic ratio and wild-type NPM1 is associated with a poor prognosis in AML. FLT3 mutations are associated with a poor prognosis in MDS. FLT3, located on 13q12, encodes for a class III receptor tyrosine kinase that regulates hematopoiesis. FLT3 mutations are found mainly in acute myeloid leukemia (30% FLT3-ITD, 7% FLT3-TKD) (PMID: 22417203). FLT3-ITD mutations are associated with an unfavorable risk status in cytogenetically normal acute myeloid leukemia. There is controversy as to whether FLT3-TKD mutations carry an equally unfavorable prognosis., The prognostic significance of FLT3 TKD mutation is less clear than that of FLT3-ITD, but it may suggest response to FLT3 inhibitors.",	


"FOXO1":	"FOXO1, located on 13q14.1, encodes for a member of the forkhead family of transcription factors, which regulate genes involved in stress resistance, metabolism, cell cycle arrest, and apoptosis, and may play an important role in tumor suppression. FOXO1 mutations are found in 12% of Burkitt lymphoma (PMID: 22885699); 9% of diffuse large B-cell lymphoma, where they are associated with an unfavorable prognosis (PMID: 23460611); and 7% of follicular lymphoma, where they are also associated an unfavorable prognosis (PMID: 26256760).",	
"GATA1":	"In patients with Down syndrome, somatic mutations in GATA1 are seen in transient abnormal myelopoiesis (TAM) and MDS/AML-associated with Down syndrome. Pathogenic GATA1 mutations result in a truncated protein which promotes megakaryocytic proliferation.",	
"GATA2":	"Myeloid neoplasms that occur on the background of a predisposing germline GATA2 mutation are categorized as myeloid neoplasms with germline predisposition in the WHO classification. GATA2 mutations are associated with a poor prognosis in MDS. GATA2, located on 3q21.3, encodes for a member of the GATA family of zinc-finger transcription factors that play an essential role in regulating transcription of genes involved in the development and proliferation of hematopoietic and endocrine cell lineages. GATA2 mutations are uncommon in sporadic myelodysplastic syndrome/acute myeloid leukemia, but they appear enriched in acute myeloid leukemia with biallelic CEBPA mutations (21%), reportedly imparting a more favorable prognosis (PMID: 23521373, PMID: 22814295). Myeloid neoplasms that occur on the background of a predisposing germline GATA2 mutation are categorized as myeloid neoplasms with germline GATA2 mutation in the 2016 revision of the WHO classification (PMID: 27069254).",
"GATA3":	"GATA3, located on 10p14, encodes for a for a member of the GATA family of zinc-finger transcription factors that play an essential role in regulating transcription of genes involved in the development and proliferation of hematopoietic and endocrine cell lineages. GATA3 mutations are found in 15% of adult T-cell leukemia/lymphoma (PMID: 26437031) and 9% of pediatric early T-cell precursor acute lymphoblastic leukemia (PMID: 22237106).",	

"GNA13":	"GNA13, located on 17q24.3, encodes for a guanine nucleotidebinding G protein involved in various transmembrane signaling pathways. GNA13 mutations are found in 24% of Burkitt lymphoma (PMID: 23143597), 20% diffuse large B-cell lymphoma, with enrichment in the germinal center B-cell-like subtype (33%) versus the activated B-cell-like subtype.",
"GNAI2":	"GNAI2, located on 3p21.31, encodes for a guanine nucleotidebinding G protein involved in various transmembrane signaling pathways. GNAI2 mutations are found in 24% of type II enteropathy-associated T-cell lymphoma (PMID: 26854024).",	
"GNAS":	"GNAS mutations are associated with a poor prognosis in MPN. GNAS, located on 20q13.3, encodes for the stimulatory alpha subunit of the G protein complex, a key component of many signal transduction pathways. GNAS mutations are found in several types of solid tumors, but are rarely reported in myeloid neoplasms (PMID: 21714648).",	
"GNB1":	"GNB1, located on 1p36.33, encodes for the beta subunit of the G protein complex, a key component of many signal transduction pathways. GNB1 mutations are infrequently found in myelodysplastic syndrome (PMID: 24220272 and PMID: 21909114).",	
"HIST1H1E":	"HIST1H1E, located on 6p21.3, encodes for a member of the histone H1 family of proteins, which interact with linker DNA between nucleosomes and functions in the compaction of chromatic into higher order structures. HIST1H1E mutations are found in 12% of follicular lymphoma (PMID: 28064239), 5% of diffuse large B-cell lymphoma (PMID: 23292937), 3% of chronic lymphocytic leukemia (PMID: 26466571), and 3% of multiple myeloma (PMID: 26282654).",	
"HRAS":	"HRAS, located on 11p15.5, encodes for a membrane protein GTPase that is a central mediator of downstream growth factor receptor signaling, critical for cell proliferation and survival. While HRAS mutations have been reported in various solid tumors, they appear to be uncommon in lymphoid neoplasms.",	
"ID3":	"ID3, located on 1p36.12, encodes for a helix-loop-helix (HLH) protein that can form heterodimers with other HLH proteins. However, the encoded protein lacks a basic DNA-binding domain and therefore inhibits the DNA binding of any HLH protein with which it interacts. ID3 mutations are found in 34-68% of Burkitt lymphoma (PMID: 23143597 and PMID: 23143595) 23% of double-hit lymphoma (PMID: 24222112), and 11% of nodal marginal zone lymphoma (PMID: 27335277).",	
"IDH1":	"Patients with IDH1-mutated AML may benefit from treatment with an IDH1 inhibitor. IDH1, located on 2q33.3, encodes for an enzyme involved in regulation of DNA methylation. IDH1 mutations induce global DNA hypermethylation, impair hematopoietic differentiation, and are found in a variety of myeloid neoplasms. IDH1 mutations appear to be rare in lymphoid neoplasms (PMID: 20667415). IDH1 mutations are associated with a poor prognosis in MPN. IDH1 mutations can be seen in various myeloid neoplasms and are associated with a poor prognosis in meloid neoplasms. Patients with IDH1-mutated AML may benefit from treatment with an IDH1 inhibitor. IDH1, located on 2q33.3, encodes for an enzyme involved in regulation of DNA methylation. IDH1 mutations induce global DNA hypermethylation and impair hematopoietic differentiation. IDH1 mutations are found in 7% of acute myeloid leukemia (PMID: 27288520), 3% of myelodysplastic syndrome (PMID: 24220272), 3% of primary myelofibrosis (PMID: 23619563). IDH1 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome (PMID: 20494930, PMID: 22033490) and primary myelofibrosis (PMID: 23619563). The prognostic significance of IDH1 mutations in acute myeloid leukemia may be related to NPM1 mutation status. IDH1 mutations in the presence of NPM1 mutations are reportedly associated with a more favorable prognosis, particularly among patients with wild-type FLT3-ITD and normal karyotype or intermediate-risk cytogenetic abnormalities (PMID: 22417203). IDH1 mutations in the absence of NPM1 mutations, however, are reportedly associated with a more unfavorable prognosis (PMID: 24443894). Patients with relapsed or refractory acute myeloid leukemia with IDH1 mutation may potentially benefit from treatment with an IDH1 inhibitor. IDH1 mutations are associated with a poor prognosis in PMF. IDH1 mutations are associated with a poor prognosis in PV.",	


"IDH2":	"Patients with IDH2-mutated AML may benefit from treatment with an IDH2 inhibitor. IDH2 mutations are associated with a poor prognosis in ET. IDH2 mutations are associated with a poor prognosis in MDS. IDH2 mutations are associated with a poor prognosis in MPN. IDH2 mutations can be seen in various myeloid neoplasms, and they are associated with a poor prognosis in MDS and MPN. Patients with IDH2-mutated AML may potentially benefit from treatment with an IDH2 inhibitor. IDH2, located on 15q26.1, encodes for an enzyme involved in regulation of DNA methylation. IDH2 mutations induce global DNA hypermethylation and impair hematopoietic differentiation. IDH2 mutations are found in 14% of acute myeloid leukemia (PMID: 27288520), 12% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 6% of chronic myelomonocytic leukemia (PMID: 27385790), 4% of myelodysplastic syndrome (PMID: 24220272), 3% of primary myelofibrosis (PMID: 23619563), 2% of polycythemia vera, and 1% of essential thrombocythemia (Tefferi A, et al. Blood Advances 2016;1:21-30)._x000D_IDH2 mutations are associated with unfavorable prognosis in myelodysplastic syndrome, primary myelofibrosis (PMID: 23619563), polycythemia vera, and essential thrombocythemia (Tefferi A, et al. Blood Advances 2016;1:21-30). The prognostic significance of IDH2 mutations in acute myeloid leukemia may be related to NPM1 mutation status. IDH2 mutations in the presence of NPM1 mutations are reportedly associated with a more favorable prognosis, particularly among patients with wild-type FLT3-ITD and normal karyotype or intermediate-risk cytogenetic abnormalities (PMID: 22417203). IDH2 mutations in the absence of NPM1 mutations, however, are reportedly associated with a more unfavorable prognosis (PMID: 24443894). Patients with relapsed or refractory acute myeloid leukemia with IDH2 mutation may potentially benefit from treatment with an IDH2 inhibitor.",	
"IKBKB":	"IKBKB, located on 8p11.2, encodes a protein that phosphorylates the inhibitor in the inhibitor/NF-kappaB complex, causing dissociation of the inhibitor and activation of NF-kappaB. IKBKB mutations are found in 7% of splenic marginal zone lymphoma (PMID: 22891273).",	
"IKZF1":	"IKZF1, located on 7p12.2, encodes for IKAROS, the founding member of a family of zinc finger transcription factors that is required for the development of all lymphoid lineages. IKZF1 mutations are found in 15-20% of pediatric B-cell acute lymphoblastic leukemia, >75% of BCR-ABL1-positive pediatric B-cell acute lymphoblastic leukemia, 50% of adult B-cell acute lymphoblastic leukemia, 65% of BCR-ABL1-positive adult B-cell acute lymphoblastic leukemia, 68% of Philadelphia chromosome-like (Ph-like) B-cell acute lymphoblastic leukemia, and 16% of BCR-ABL1-negative non-Ph-like B-cell acute lymphoblastic leukemia. IKZF1 mutations are associated an unfavorable prognosis in B-cell acute lymphoblastic leukemia.",	
"IKZF3":	"IKZF3, located on 17q12-q21.1, encodes for a member of the Ikaros family of zinc-finger proteins, which are involved in the regulation of B-cell proliferation and differentiation. IKZF3 mutations are found in 13% of near-haploid pediatric acute lymphoblastic leukemia (PMID: 23334668) and 2% of chronic lymphocytic leukemia (PMID: 26466571).",	
"IRAK4":	"IRAK4, located on 12q12, encodes for a kinase that activates NF-kappaB in both the toll-like receptor and T-cell receptor signaling pathways. While alterations of these pathways are important in the pathogenesis of various B-cell neoplasms, IRAK4 mutations appear to be uncommon.",	
"ITPKB":	"ITPKB, located on 1q42.12, encodes for a protein that regulates inositol phosphate metabolism by phosphorylation of second messenger inositol 1,4,5-trisphosphate to Ins(1,3,4,5)P4. The activity of this protein is responsible for regulating the levels of a large number of inositol polyphosphates that are important in cellular signaling. ITPKB alterations including mutations and copy number alterations are uncommon in lymphoid neoplasms.",
"JAK1":	"JAK1, located on 1p31.3, encodes for one of the four members of the JAK family of proteins that also includes JAK2, JAK3 and TYK2. These membrane-associated, intracellular, non-receptor tyrosine kinase proteins mediate signals initiated by cytokines and growth factor receptors. Their activation leads to subsequent binding, phosphorylation and activation of STAT, MAP kinase/ERK and PI3 kinase/AKT signaling pathways. JAK1 mutations are found in 18% of adult T-cell acute lymphoblastic leukemia (PMID: 18362173), 15% of systemic ALK-negative anaplastic large cell lymphoma (PMID: 25873174), 8% of T-cell prolymphocytic leukemia (PMID: 24825865), and a subset of Ph-like B-cell acute lymphoblastic leukemia (PMID: 25207766).",	
"JAK2":	"JAK2 exon 12-mutated PV is associated with younger age, increased mean hemoglobin/hematocrit, and lower mean white blood cell and platelet counts at diagnosis compared to JAK2 V617F-mutated PV. However, both JAK2 mutations are associated with similar rates of thrombocytosis, evolution to myelofibrosis or leukemia, and death. JAK2, located on 9p24.1, encodes for one of the four members of the JAK family of proteins that also includes JAK1, JAK2 and TYK2. These membrane-associated, intracellular, non-receptor tyrosine kinase proteins mediate signals initiated by cytokines and growth factor receptors. Their activation leads to subsequent binding, phosphorylation and activation of STAT, MAPK/ERK and PI3K/AKT signaling pathways. JAK2 V617F mutations are commonly found in myeloproliferative neoplasms. In regards to lymphoid neoplasms, gains in chromosome 9p24 involving the JAK2 gene are found in 50-75% of primary mediastinal large B-cell lymphoma, while JAK2 rearrangements are found in 7% of Ph-like B-cell acute lymphoblastic leukemia (PMID: 25207766). JAK2 mutations are frequently seen in MPN and occasionally seen in MDS/MPN. JAK2 mutations are included as one of the major diagnostic criteria for MPN in the WHO classification. JAK2 V617F mutation is associated with an intermediate prognosis and higher risk of thrombosis in PMF compared with CALR mutation.",	
	
"JAK3":	"JAK3, located on 19p13.11, encodes for one of the four members of the JAK family of proteins that also includes JAK1, JAK2 and TYK2. These membrane-associated, intracellular, non-receptor tyrosine kinase proteins mediate signals initiated by cytokines and growth factor receptors. Their activation leads to subsequent binding, phosphorylation and activation of STAT, MAPK/ERK and PI3K/AKT signaling pathways. JAK3 mutations are found in various T-cell neoplasms including 35% of type II enteropathy-associated T-cell lymphoma (PMID: 26854024), 30% of T-cell prolymphocytic leukemia (PMID: 24825865), 21% of extranodal NK/T-cell lymphoma nasal type (PMID: 23689514), 11% of adult T-cell leukemia/lymphoma (PMID: 21821710), 10% of T-cell acute lymphoblastic leukemia (PMID: 23263491), and 8% of mycosis fungoides (PMID: 26082451).",	
"KDM6A":	"KDM6A mutations have been reported in association with longer overall survival in MDS patients treated with azacitidine.",	
"KIT":	"Core binding factor AML with t(8;21) and KIT mutation is associated with decreased remission duration and overall survival. The prognostic association of KIT mutation in core binding factor AML with inv(16), however, is less clear. Activating point mutations at codon 816 of KIT are frequently seen in mastocytosis and are included as one of the minor diagnostic criteria for systemic mastocytosis in the WHO classification. KIT mutations can be seen in various myeloid neoplasms. Activating point mutations at codon 816 of KIT are frequently seen in mastocytosis, and are included as one of the minor diagnostic criteria for systemic mastocytosis in the WHO classification. KIT mutations can also be seen in core binding factor AML, and they are associated with decreased remission duration and overall survival in AML with t(8;21). The prognostic association of KIT mutations in AML with inv(16), however, is less clear. KIT, located on 4q12, encodes for a tyrosine kinase which leads to downstream signaling pathways that are involved in proliferation, differentiation, and survival. KIT mutations are found in 75% of systemic mastocytosis (PMID: 27214377), and are included as one of the minor diagnostic criteria for systemic mastocytosis in the WHO classification. KIT mutations are also found in 4% of acute myeloid leukemia, but 26% of acute myeloid leukemia with core binding factor rearrangements [inv(16), t(16;16), or t(8;21)] (PMID: 27288520). Core binding factor acute myeloid leukemias with KIT mutation are associated with higher risk of relapse and an intermediate risk status.",

	
"KLF2":	"KLF2, located on 19p13.11, encodes for a member of the Kruppel-like family of zinc finger transcription factors. KLF2 mutations are found in 42% of splenic marginal zone lymphoma, where they are associated with IGHV1-2 usage and 7q deletion (PMID: 25428260); 17% of nodal marginal zone lymphoma (PMID: 27335277), 16% of hairy cell leukemia, 11% of diffuse large B-cell lymphoma, 8% of extranodal marginal zone lymphoma, 4% of follicular lymphoma, 4% of mantle cell lymphoma, and 3% of chronic lymphocytic leukemia (PMID: 25283840).",	
"KMT2D":	"KMT2D (also known as MLL2), located on 12q13.12, encodes for a histone methyltransferase that methylates the Lys-4 position of histone H3. Alterations of KMT3D appear to promote lymphomagenesis by disrupting germinal center B-cell development (PMID: 26366712). KMT2D mutations are found in 79% of follicular lymphoma (PMID: 26256760), 34% of nodal marginal zone lymphoma (PMID: 27335277), 32% of diffuse large B-cell lymphoma (PMID: 21796119), 20% of mantle cell lymphoma (PMID: 24682267), 15% of splenic marginal zone lymphoma (PMID: 22891273), 12% of adult T-cell acute lymphoblastic leukemia (PMID: 25595890), 7% of Waldenstrom macroglobulinemia (PMID: 24366360), and 7% of NK/T-cell lymphoma (PMID: 26192917).",	
"KRAS":	"KRAS, located on 12p12.1, encodes for a membrane protein GTPase that is a central mediator of downstream growth factor receptor signaling, critical for cell proliferation and survival. KRAS mutations are found in 24% of multiple myeloma (PMID: 24335104); 15% of relapsed childhood B-cell acute lymphoblastic leukemia, where they are associated with an unfavorable prognosis (PMID: 25253770); and 6% of chronic lymphocytic leukemia, where they are associated with an unfavorable prognosis independent of other prognostic factors (PMID: 27226433). Somatic mutations in KRAS are included as one of the diagnostic criteria for JMML in the WHO classification. KRAS mutations are associated with a poor prognosis in MPN. KRAS, located on 12p12.1, encodes for a membrane protein GTPase that is a central mediator of downstream growth factor receptor signaling, critical for cell proliferation and survival. KRAS mutations are found in 14% of juvenile myelomonocytic leukemia (PMID: 23832011), 9% of chronic myelomonocytic leukemia (PMID: 27385790), 8% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 6% of acute myeloid leukemia (PMID: 27288520), 3% of myelodysplastic syndrome (PMID: 24220272), and 2% of primary myelofibrosis (PMID: 24150215).",		
"MALT1":	"MALT1, located on 18q21, encodes for a caspase-like cysteine protease that is essential for NF-kappaB activation downstream of cell surface receptors. MALT1 is recurrently rearranged in extranodal marginal zone lymphoma of mucosa-associated lymphoid tissue (MALT lymphoma). The t(11;18)(q21;q21) results in formation of the chimeric fusion gene, API2-MALT1, and is found in approximately 30% of MALT lymphoma, most commonly of the stomach and lung. The t(14;18)(q32;q21) results in deregulated expression of MALT1, and is found in 10-15% of MALT lymphoma, most commonly of the liver, skin, ocular adnexa, and salivary gland. Gain of the chromosome region 18q21, which includes MALT1, is found in 38% of diffuse large B-cell lymphoma, where it is more commonly seen in the activated B-cell-like subtype and may be associated with a more unfavorable prognosis (PMID: 18367485).",	
"MAP2K1":	"MAP2K1, located on 15q22.1-q22.33, encodes for a member of the MAP kinase family, which is involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. MAP2K1 mutations are found in 43% of pediatric-type nodal follicular lymphoma (PMID: 27325104), 42% of hairy cell leukemia variant, and 22% of classical hairy cell leukemia (71% of IGHV4-34-positive and 5% of IGHV4-23-negative subtype) (PMID: 24241536), 28% of Langerhans cell histiocytosis (PMID: 24982505), 11% of splenic diffuse red pulp small B-cell lymphoma (PMID: 26426381), and 2% chronic lymphocytic leukemia (PMID: 23415222).",	
"MAP3K14":	"MAP3K14, located on 17q21.3, encodes for the mitogen-activated protein kinase kinase kinase 14, which is a serine/threonine protein-kinase that binds to TRAF2 and stimulates NF-kappaB activity. Alterations of MAP3K14 including gains or mutations are found in 31% of classical Hodgkin lymphoma (PMID: 22469134) and 8% of splenic marginal zone lymphoma (PMID: 21881048).",	
"MAPK1":	"MAPK1, located on 22q11.21, encodes for a member of the MAP kinase family, which is involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. MAPK1 mutations are found in 9% of pediatric-type nodal follicular lymphoma (PMID: 27325104) and 3% of chronic lymphocytic leukemia (PMID: 22150006).",	
"MED12":	"MED12, located on Xq13, encodes for a subunit of the Mediator multiprotein complex that functions in transcription activation or repression depending on the factors with which it interacts. MED12 mutations are found in 5% of chronic lymphocytic leukemia, and are associated with unfavorable prognostic markers such unmutated IGHV and ZAP70 expression (PMID: 25595892).",	
"MEF2B":	"MEF2B, located on 19p13.11, encodes for a member of the MADS/MEF2 family of DNA binding proteins, which are thought to regulate gene expression, including expression of the smooth muscle myosin heavy chain gene. MEF2B mutations found in 18% of diffuse large B-cell lymphoma (PMID: 22343534); 10% of follicular lymphoma, where they are associated with a more favorable prognosis (PMID: 26256760); and 3% of mantle cell lymphoma (PMID: 24145436).",	
"MPL":	"MPL mutation status is included as one of the major diagnostic criteria for ET in the WHO classification. MPL mutations are frequently seen in MPN and occasionally seen in MDS/MPN. MPL mutations are included as one of the major diagnostic criteria for ET and PMF in the WHO classification. MPL, located on 1p34, encodes for the thrombopoietin receptor, an important growth and survival factor for megakaryocytes. MPL mutations are found in 5% of primary myelofibrosis (PMID: 23619563) and 4% of essential thrombocythemia (PMID: 24366362), and are includes as one of the major diagnostic criteria for primary myelofibrosis and essential thrombocythemia in the 2016 revision of the WHO classification (PMID: 27069254). MPL mutations are included as one of the major diagnostic criteria for ET in the WHO classification. MPL W515L/K is associated with an intermediate prognosis and higher risk of thrombosis in PMF compared with CALR mutation. MPL mutations are frequently seen in MPN, while SF3B1 mutations are frequently seen in MDS with ring sideroblasts. Co-mutation of MPL and SF3B1 is suggestive of a MDS/MPN-RS-T.",	
"MYC":	"MYC, located on 8q24.21, encodes for a multifunctional nuclear phosphoprotein that plays a role in cell cycle progression, apoptosis, and cellular transformation. This protein also functions as a transcription factor that controls gene expression. MYC translocations to one of the immunoglobulin loci are the cytogenetic hallmark of Burkitt lymphoma, although MYC mutations are also reportedly present in 40% of cases (PMID: 23143597). Concurrent MYC and BCL2 or BCL6 rearrangement defines a subset of high-grade B-cell lymphomas known as double-hit lymphomas, which are associated with an unfavorable prognosis and poor response to conventional therapy.",

	
"MYCN":	"MYCN, located on 2p24.3, is a member of the MYC family of oncogenes, and it plays an important role in regulating cell proliferation and apoptosis. MYCN mutations or amplification appear to be uncommon in lymphoid neoplasms.",	
"MYD88":	"MYD88, located on 3p22, encodes for a protein that functions as a signal transducer in the IL-1 and TLR signaling pathways, thus, playing a crucial role in both innate and adaptive immune response. MYD88 mutations are found in 89% of Waldenstrom macroglobulinemia (PMID: 25853747), 79% of primary central nervous system lymphoma (PMID: 25991819), 29% of the activated B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 21179087), 5-15% of splenic marginal zone lymphoma (PMID: 22891273 and PMID: 25723115), 10% of chronic lymphocytic leukemia (PMID: 22150006), 9% of nodal marginal zone lymphoma (PMID: 27335277), and 5% of Burkitt lymphoma (PMID: 22885699). The mutation status of MYD88 and CXCR4 in Waldenstrom macroglobulinemia may help predict responsiveness to the BTK inhibitor ibrutinib, with response rates reportedly highest among patients with mutated MYD88 and wild-type CXCR4, followed by patients with mutated MYD88 and CXCR4, and patients with wild-type MYD88 and CXCR4 (PMID: 25853747). MYD88 mutations are associated with younger age, mutated IGHV, 13q deletion, low CD38 and ZAP70 expression, and a more favorable prognosis in chronic lymphocytic leukemia (PMID: 24782504).",	
"NF1":	"NF1, located on 17q11.2, encodes for a tumor suppressor protein that suppresses the function of the RAS protein, which promotes cell growth and differentiation. Germline NF1 mutations are seen in neurofibromatosis type 1, which is associated with predisposition to several types of neoplasms. About 10-15% of juvenile myelomonocytic leukemia arise in children with neurofibromatosis type 1 and/or showing somatic inactivation of NF1 (PMID: 19571318). Somatic NF1 mutations are found in 3% of myelodysplastic syndrome (PMID: 24220272) and 3% of chronic myelomonocytic leukemia (PMID: 27385790). Germline NF1 mutations are seen in neurofibromatosis type 1, and are associated with predisposition to several types of neoplasms, including JMML. NF1 mutations are included as one of the diagnostic criteria for JMML and are associated with a poor prognosis.",	
"NFKBIE":	"NFKBIE, located on 6p21.1, encodes for a protein that binds to components of NF-kappaB, trapping the complex in the cytoplasm and preventing it from activating genes in the nucleus. NKFBIE mutations, predominantly truncating deletions, are found in 27% of classical Hodgkin lymphoma, 23% of primary mediastinal B-cell lymphoma, 11% of small lymphocytic lymphoma, 4% of mantle cell lymphoma, 4% of diffuse large B-cell lymphoma, 3% of primary central nervous system, 2% of splenic marginal zone lymphoma, 1% of follicular lymphoma, and 1% of T-cell acute lymphoblastic leukemia (PMID: 27670424). NFKIE mutations are associated with an unfavorable prognosis in primary mediastinal B-cell lymphoma (PMID: 27670424) and chronic lymphocytic leukemia (PMID: 25987724).",	
"NOTCH1":	"NOTCH1, located on 9q34.3, encodes for a ligand-activated transmembrane protein that regulates downstream pathways important for cell growth. NOTCH1 mutations are found in 56% of T-cell acute lymphoblastic leukemia (PMID: 15472075), 15% of adult T-cell leukemia/lymphoma (PMID: 26437031), 12% of chronic lymphocytic leukemia (PMID: 23860447), 12% of mantle cell lymphoma (PMID: 22210878), 8% of Burkitt lymphoma (PMID: 23143597), and 5% of splenic marginal zone lymphoma (PMID: 22891273). _x000D_NOTCH1 mutations correlate with unmutated IGHV, trisomy 12, and high CD38 and ZAP70 expression, and are associated with an unfavorable prognosis in chronic lymphocytic leukemia and reduced benefit from the addition of rituximab to fludarabine and cyclophosphamide chemotherapy (PMID: 23086750 and PMID: 24652989). NOTCH1 mutations are also associated with an unfavorable prognosis in mantle cell lymphoma (PMID: 22210878), but favorable prognosis in adult T-cell acute lymphoblastic leukemia in the absence of RAS or PTEN abnormalities (PMID: 24166518).",	


"NOTCH2":	"NOTCH2, located on 1p13-p11, is a paralogue of NOTCH1, implicated in several phases of lymphoid development, including terminal B-cell differentiation into immunoglobulin-secreting cells. NOTCH2 mutations are predicted to cause impaired degradation of the NOTCH2 protein by eliminating the C-terminal PEST domain, which is required for proteasomal recruitment. NOTCH2 mutations are found in 21% of splenic marginal zone lymphoma (PMID: 22891273), 20% of nodal marginal zone lymphoma (PMID: 27335277); 5% of mantle cell lymphoma, where they are associated with blastoid/pleomorphic morphology and unfavorable prognosis (PMID: 24145436); and 3% of Waldenstrom macroglobulinemia (PMID: 24366360).",	
"NOTCH3":	"NOTCH3, located on 19p13.12, encodes for a ligand-activated transmembrane protein that plays a key role in the function and survival of vascular smooth muscle cells. While amplification, and to a lesser degree, mutations of NOTCH3 have been reported in various solid tumors, they appear to be uncommon in lymphoid neoplasms, with activating NOTCH3 mutations being reported in a few cases of T-cell acute lymphoblastic leukemia (PMID: 27157619).",	
"NPM1":	"NPM1 mutations without FLT3 ITD with high allelic ratio are associated with a favorable prognosis in AML. NPM1 mutations are associated with a poor prognosis in MDS. NPM1 mutations without FLT3 ITD are associated with a favorable prognosis in AML. NPM1 mutations can also be seen in a small subset of MDS, where they are associated with a poor prognosis.NPM1, located on 5q35.1, encodes for a nuclear protein involved in the regulation of tumor suppressor activity. NPM1 mutations are found in 33% of acute myeloid leukemia (PMID: 27288520), 20% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), and 2% of myelodysplastic syndrome (PMID: 21714648). Acute myeloid leukemia with mutated NPM1 is categorized as an acute myeloid leukemia with recurrent genetic abnormality in the 2016 revision of the WHO classification (PMID: 27069254). NPM1 mutations are associated with a favorable risk status in the absence of FLT3-ITD mutations in cytogenetically normal acute myeloid leukemia, and reportedly predict an improved outcome with high-dose induction chemotherapy compared with standard-dose induction chemotherapy (PMID: 22417203).",	
"NRAS":	"NRAS, located on 1p13.2, encodes for a membrane protein GTPase that is a central mediator of downstream growth factor receptor signaling, critical for cell proliferation and survival. NRAS mutations are found in 20% of multiple myeloma, where they are associated with reduced sensitivity to bortezomib but not high-dose dexamethasone (PMID: 24335104); 15% of relapsed B-cell acute lymphoblastic leukemia (PMID: 25253770), and 2% of chronic lymphocytic (PMID: 26466571). NRAS mutations are associated with a poor prognosis in CMML. Somatic mutations in NRAS are included as one of the diagnostic criteria for JMML in the WHO classification. NRAS, located on 1p13.2, encodes for a membrane protein GTPase that is a central mediator of downstream growth factor receptor signaling, critical for cell proliferation and survival. NRAS mutations are found in 20% of multiple myeloma, where they are associated with reduced sensitivity to bortezomib but not high-dose dexamethasone (PMID: 24335104); 15% of relapsed B-cell acute lymphoblastic leukemia (PMID: 25253770), and 2% of chronic lymphocytic (PMID: 26466571). NRAS mutations are associated with a poor prognosis in MDS, particularly in patients predicted to have lower-risk MDS. NRAS mutations are associated with a poor prognosis in MPN.NRAS mutations can be seen in various myeloid neoplasms and are associated with poor prognosis in MDS, particularly in patients predicted to have lower-risk MDS, MPN, and CMML.",	
"NT5C2":	"NT5C2, located on 10q24.32-q24.33, encodes for a hydrolase that serves as an important role in cellular purine metabolism by acting primarily on inosine 5'-monophosphate and other purine nucleotides. NT5C2 mutations are found in 19% of relapsed T-cell acute lymphoblastic leukemia and 3% of relapsed B-cell acute lymphoblastic leukemia, where they are associated with resistance to chemotherapy with 6-mercaptopurine and 6-thioguanine (PMID: 23377281).",	
"P2RY8":	"P2RY8, located on Xp22.33 and Yp11, encodes for a G protein-coupled purinergic receptor whose normal function has not been extensively characterized. P2RY8 is most notable for its involvement in a chromosomal translocation with CRLF2 in 7% of B-cell acute lymphoblastic leukemia and 53% of Down syndrome-associated acute lymphoblastic leukemia (PMID: 19838194). P2YR8 are also found in 32% of Burkitt lymphoma, 4% of the germinal center B-cell-like subtype of diffuse large B-cell lymphoma, and 3% of the activated B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 25274307).",	
"PDGFRA":	"PDGFRA, located on 4q12, encodes for a cell surface tyrosine kinase receptor for members of the platelet-derived growth factor family. In the WHO classification, PDGFRA rearrangements define a category of disease called myeloid/lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA. This entity can present as chronic eosinophilic leukemia, acute myeloid leukemia, or T-lymphoblastic leukemia, and is associated with responsiveness to tyrosine kinase inhibitors.",	
"PDGFRB":	"PDGFRB encodes a tyrosine kinase receptor for members of the platelet-derived growth factor family. PDGFRB rearrangements define a category of disease called myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRB. This entity can present as chronic myelomonocytic leukemia, atypical chronic myeloid leukemia, chronic eosinophilia leukemia, acute myeloid leukemia, and juvenile myelomonocytic leukemia, and is associated with responsiveness to tyrosine kinase inhibitors. PDGFRB rearrangements are also found in 6% of Ph-like B-cell acute lymphoblastic leukemia (PMID: 25207766).",	
"PHF6":	"PHF6 mutations can be seen in various myeloid neoplasms. In MDS, PHF6 mutations are more frequently seen with excess blasts, but they show no association with survival. In MPN, PHF6 mutations are associated with a poor prognosis. PHF6 mutations are more frequently seen in MDS with excess blasts, but they show no association with survival. PHF6 mutations are associated with a poor prognosis in MPN. PHF6, located on Xq26.3, encodes for a protein with two PHD-type zinc finger domains that localizes to the nucleolus, indicating a potential role in transcriptional regulation. PHF6 mutations are found in 3% of myelodysplastic syndrome (PMID: 24220272); and 2% of acute myeloid leukemia, where they are associated with a male preponderance and an unfavorable prognosis in patients with normal karyotype or intermediate-risk cytogenetic abnormalities (PMID: 21030981, PMID: 22417203).",	

"PIK3CA":	"PIK3CA, located on 3q26.32, encodes for a subunit of PI3K. The PI3K pathway is activated in a variety of neoplasms, either by activating mutations of PI3K signaling modules or by activation of upstream surface receptors. PIK3CA mutations are found in 8% of diffuse large B-cell lymphoma (PMID: 17657213).",	
"PIK3CD":	"PIK3CD, located on 1p36.22, encodes for a subunit of PI3K. The PI3K pathway is activated in a variety of neoplasms, either by activating mutations of PI3K signaling modules or by activation of upstream surface receptors. PIK3CD mutations are found in 9% of hepatosplenic T-cell lymphoma (PMID: 28122867).",	
"PIK3R1":	"PIK3R1, located on 5q13.1, encodes for a subunit of PI3K. The PI3K pathway is activated in a variety of neoplasms, either by activating mutations of PI3K signaling modules or by activation of upstream surface receptors. PIK3R1 mutations are found in 14% of angioimmunoblastic and other follicular helper T-cell-derived lymphomas (PMID: 27369867), 5% of Burkitt lymphoma (PMID: 23143597).",	
"PIM1":	"PIM1, located on 6p21.2, encodes for a kinase involved in cell cycle regulation, anti-apoptotic activity, and homing and migration of receptor tyrosine kinases mediated via the JAK/STAT pathway. PIM1 mutations are found in 13% follicular lymphoma, 46% of transformed follicular lymphoma, 12-33% of de novo diffuse large B-cell lymphoma, and 50% of primary central nervous system lymphoma (PMID: 24388756, PMID: 23292937, PMID: 22343534, PMID: 21804550 and PMID: 24240734].",	
"PLCG1":	"PLCG1, located on 20q12, encodes a protein that plays a central role in the downstream signaling elicited by the T-cell receptor. PLCG1 mutations are found in various T-cell neoplasms, including 36% of adult T-cell leukemia/lymphoma (PMID: 26437031), 18% of Sezary syndrome (PMID: 26551670), 15% of peripheral T-cell lymphoma not otherwise specified (PMID: 25304611), and 14% of angioimmunoblastic and other follicular helper T-cell-derived lymphomas (PMID: 27369867).",	
"PLCG2":	"PLCG2, located on 16q24.1, encodes for a kinase that acts immediately downstream of BTK in the BCR signaling pathway. Gain of function mutations in PLCG2, particularly the S707Y, R665W, and L845F variants, reportedly lead to autonomous B-cell receptor activity independent of BTK, and are associated with resistance to the BTK inhibitor ibrutinib in patients with chronic lymphocytic leukemia (PMID: 24869598 and PMID: 25972157).",	
"POT1":	"POT1, located on 7q31.33, encodes for a nuclear protein involved in telomere maintenance. POT1 mutations are found in 8% of chronic lymphocytic leukemia, and are associated with an unfavorable prognosis independent of other prognostic factors (PMID: 27226433). POT1 mutations are also found in 10% of adult T-cell leukemia/lymphoma (PMID: 26437031) and 5% of mantle cell lymphoma (PMID: 24682267).",	
"PPM1D":	"Somatic mutations in PPM1D can be found in patients over the age of 50 without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals lack cytopenias and have only a single mutation with low allele frequency . PPM1D, located on 17q23.2, encodes for a member of the PP2C family of serine/threonine protein phosphatases, which negatively regulate cellular responses to environmental stress, in part by inhibiting TP53 activity. PPM1D mutations are more commonly found in patients with therapy-related myelodysplastic syndrome than primary myelodysplastic syndrome (15% versus 3%), and are associated with shorter survival after hematopoietic stem cell transplantation (PMID: 28177873). PPM1D mutations can also be found in clonal hematopoiesis of indeterminate potential (CHIP), which is an age-related condition characterized by somatic mutations in the peripheral blood and/or bone marrow of otherwise healthy adults. In patients undergoing autologous stem cell transplantation for lymphoma, CHIP at the time of transplantation, particularly if PPMD1 is mutated, is reportedly associated with inferior survival and increased risk of therapy-related myeloid neoplasm (PMID: 28068180). PPM1D mutations are associated with therapy-related MDS, but are not associated with a poor prognosis independent of TP53. PPM1D mutations can be seen in various myeloid neoplasms and are associated with therapy-related MDS, but are not associated with a poor prognosis independent of TP53. PPMD1 mutations can also be commonly seen in CHIP and CCUS.",	
"PRDM1":	"PRDM1 (also known as BLIMP1), located on 6q21, encodes for a transcriptional repressor involved in terminal B-cell and T-cell differentiation (PMID: 16492805 and PMID: 25115512). PRDM1 is inactivated by multiple mechanisms, including deletions, mutations, and transcriptional repression by constitutively active BCL6, in 53% of the activated B-cell-like subtype of diffuse large B-cell lymphoma. PRDM1 alterations are infrequent in the germinal center B-cell-like subtype of diffuse large B-cell lymphoma and follicular lymphoma (PMID: 21156281). Homozygous PRDM1 deletions or mutations are associated with an unfavorable prognosis in the activated B-cell-like but not the germinal center B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 27568520). PRDM1 is also inactivated by deletion or mutations in 39% of ALK-negative and 3% of ALK-positive anaplastic large cell lymphoma (PMID: 24004669) and 19% of primary central nervous system lymphoma (PMID: 18596541).",	
"PRPS1":	"PRPS1, located on Xq22.3, encodes for an enzyme that catalyzes the phosphoribosylation of ribose 5-phosphate to 5-phosphoribosyl-1-pyrophosphate, which is necessary for purine metabolism and nucleotide biosynthesis. PRPS1 mutations are found in 7% of relapsed pediatric B-cell acute lymphoblastic leukemia, and are associated with thiopurine resistance (PMID: 25962120).",	
"PTEN":	"PTEN (phosphatase and tensin homolog deleted on chromosome ten) encodes for a lipid/protein phosphatase that plays a role in multiple cell processes, including growth, proliferation, survival, and maintenance of genomic integrity. PTEN negatively regulates intracellular levels of phosphatidylinositol-3,4,5-trisphosphate in cells and functions as a tumor suppressor by negatively regulating AKT/PKB signaling pathway. PTEN is frequently deleted or mutated in a wide variety of neoplasms. PTEN activity can also be lost through other mechanisms such as epigenetic changes or post-translational modifications. Among lymphoid neoplasms, PTEN deletions or mutations are found in 36% of Sezary syndrome (PMID: 24062018), 23% of mycosis fungoides (PMID: 10779442); 12% of adult T-cell acute lymphoblastic leukemia, where they are associated with an unfavorable prognosis (PMID: 24166518); 10% of the germinal center B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 25499448), and 7% of Burkitt lymphoma (PMID: 22885699).",	
"PTPN11":	"PTPN11 mutations are associated with a poor prognosis in MDS after hypomethylating agent treatment, but do not predict response. Somatic mutations in PTPN11 are included as one of the diagnostic criteria for JMML in the WHO classification, and are associated with a poor prognosis. PTPN11, located on 12q24.13, encodes for the SHP2 protein tyrosine phosphatase that regulates growth factor-dependent proliferation and differentiation in hematopoietic progenitors. PTPN11 mutations are found in a variety of myeloid neoplasms and also uncommonly in lymphoid neoplasms, including 4% of relapsed childhood B-cell acute lymphoblastic leukemia (PMID: 25253770) and 1% of chronic lymphocytic leukemia (PMID: 26466571). PTPN11, located on 12q24, encodes for a member of the protein tyrosine phosphatase family, which includes signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. PTPN11 mutations are found in 42% of juvenile myelomonocytic leukemia (PMID: 23832011), 10% of acute myeloid leukemia (PMID: 27288520), 2% of chronic myelomonocytic leukemia (PMID: 27385790); and 4% of myelodysplastic syndrome, where they are associated with an unfavorable prognosis (PMID: 25224413).",








	
"RAD21":	"RAD21 mutations are associated with a poor prognosis in MDS, and are more frequently seen in patients with high IPSS scores and secondary AML. RAD21, along with SMC1A, SMC3, and STAG2, encode for subunits of the cohesin complex, which is involved in DNA double-strand break repair and chromatid segregation during mitosis. Cohesin complex mutations are generally mutually exclusive of each other and are found in 20% of secondary acute myeloid leukemia, 17% of higher-risk myelodysplastic syndrome, 11% of lower-risk myelodysplastic syndrome, 11% of de novo acute myeloid leukemia, 7% of myeloproliferative neoplasm, and 4% of myelodysplastic/myeloproliferative neoplasm. Cohesin complex mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, and are more frequently found in patients with high IPSS scores and secondary acute myeloid leukemia (PMID: 25006131).",	
"RB1":	"Loss of function of the RB1 gene, located on 13q14, is not only related to the development of retinoblastoma, but may also be important in the initiation and/or progression of other neoplasms. Deletions of 13q with loss of RB1 are found in 55% of chronic lymphocytic leukemia, where they are associated with a favorable prognosis as a sole abnormality (PMID: 11136261); 26% of younger mantle cell lymphoma patients, where they are associated with an unfavorable prognosis (PMID: 26022239); and 16% of Sezary syndrome (PMID: 26551667).",
"REL":	"REL, located on 2p16.1, encodes for a member of the NF-kappaB family of transcription factors. Gains or amplification of REL are found in approximately 75% of primary mediastinal B-cell lymphoma (PMID: 17243160), 50% of classical Hodgkin lymphoma (PMID: 12511414); and 7% of diffuse large B-cell lymphoma, with enrichment in the germinal center B-cell-like subtype (PMID: 12075054).",	
"RHOA":	"RHOA, located on 3p21.31, encodes for a member of the Rho family of small GTPases, which cycle between inactive GDP-bound and active GTP-bound states and function as molecular switches in signal transduction cascades. Overexpression of RHOA is associated with tumor proliferation and metastasis. RHOA mutations are found in 71% of angioimmunoblastic T-cell lymphoma, 62% of peripheral T-cell lymphoma not otherwise specified with follicular helper T-cell-like immunophenotype, 0% of peripheral T-cell lymphoma not otherwise specified without follicular helper T-cell-like immunophenotype (PMID: 24413737), 20% of endemic Burkitt lymphoma (PMID: 26468873), 15% of adult T-cell leukemia/lymphoma (PMID: 26574607), 9% of pediatric Burkitt lymphoma (PMID: 25044415), 7% of Sezary syndrome (PMID: 26551670).",
"RIPK1":	"RIPK1, located on 6p25.2, encodes for a kinase involved in a variety of cellular pathways including the NF-kappaB pathway and programmed necrotic cell death. RIPK1 mutations are uncommon in lymphoid neoplasms, and are found in 3% of chronic lymphocytic leukemia (PMID: 23415222).",	
"RPS15":	"RPS15, located on 19p13.3, encodes for a ribosomal protein that is a component of the 40S subunit. RPS15 mutations are found in 20% of fludarabine, cyclophosphamide, and rituximab (FCR) relapsed chronic lymphocytic leukemia, and are likely to be early clonal events that confer an unfavorable prognosis (PMID: 26675346).",	
"RUNX1":	"RUNX1 mutations are associated with a poor prognosis in AML. RUNX1 mutations are associated with a poor prognosis in CMML. Myeloid neoplasms that occur on the background of a predisposing germline RUNX1 mutation are categorized as myeloid neoplasms with germline predisposition in the WHO classification. RUNX1, located on 21q22.12, encodes for the alpha subunit of core binding factor and is thought to be involved in the development of normal hematopoiesis. RUNX1 mutations are found in a variety of myeloid neoplasms. Among lymphoid neoplasms, the t(12;21)(p13;q22), resulting in the ETV6-RUNX1 gene fusion, is found in 22% of pediatric and 2% of adult B-cell acute lymphoblastic leukemia, and is associated with a favorable prognosis. RUNX1 mutations are associated with a poor prognosis in MDS. RUNX1 mutations are associated with a poor prognosis in MPN. RUNX1 mutations can be seen in various myeloid neoplasms, and are associated a poor prognosis in MDS, AML, MPN, and CMML. RUNX1, located on 21q22.3, encodes for the alpha subunit of core binding factor and is thought to be involved in the development of normal hematopoiesis. RUNX1 mutations are found in 15% of acute myeloid leukemia (PMID: 27288520), 11% of myelodysplastic syndrome (PMID: 24220272), 8% of chronic myelomonocytic leukemia (PMID: 27385790), 3% of systemic mastocytosis (PMID: 27214377), 2% of essential thrombocythemia, and 2% of polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30). Myeloid neoplasms that occur on the background of a predisposing germline RUNX1 mutation are categorized as myeloid neoplasms with germline RUNX1 mutation in the 2016 revision of the WHO classification (PMID: 27069254). RUNX1 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, independent of IPSS, IPSS-R, age, and other gene mutations. When mutation status is integrated into the survival analysis by IPSS risk categories, RUNX1 mutations are reported to shift the survival curve of the IPSS risk category to resemble that of the next highest IPSS risk level (PMID: 21714648). Accordingly, current guidelines state that gene mutation analysis can refine the prognosis of myelodysplastic syndrome in patients risk stratified by the IPSS or IPSS-R, and may be helpful in patients predicted to have intermediate risk. RUNX1 mutations are independently associated with unfavorable outcomes and shorter survival after hematopoietic stem cell transplantation in patients with myelodysplastic syndrome and myelodysplastic syndrome/acute myeloid leukemia (PMID: 27601546). RUNX1 mutations are also associated with an unfavorable prognosis in acute myeloid leukemia (PMID: 27288520), chronic myelomonocytic leukemia (PMID: 27385790), and systemic mastocytosis (PMID: 27214377).",	
"S1PR2":	"S1PR2, located on 19p13.2, encodes for a receptor that binds to sphingosine-1-phosphate (S1P), a lipid that can promote migration of lymphocytes. Depending on the receptor expressed, S1P signaling can promote retention of B-cells in germinal centers or the egress from lymphoid tissues into circulation. S1P2 is highly expressed in germinal center B-cells, and coupled with GNAI2 and GNA13, promotes retention in the germinal centers. S1PR2 mutations are found in 4-8% of diffuse large B-cell lymphoma, with enrichment in the germinal center B-cell-like subtype (PMID: 23699601 and PMID: 21804550); and 3% of Burkitt lymphoma (PMID: 25274307).",	
"SAMHD1":	"SAMHD1, located on 20pter-q12, encodes for a protein that degrades dNTPs into their deoxynucleosides and inorganic triphosphates. SAMHD1 activity regulates the intracellular dNTP pool, which is required for DNA polymerase functioning and DNA synthesis (PMID: 24526775). Germline SAMHD1 mutations have been described in patients with Aicardi-Goutieres syndrome, a congenital autoimmune disease. Somatic SAMHD1 mutations are found in 4% of chronic lymphocytic leukemia and up to 11% of relapsed/refractory chronic lymphocytic leukemia (PMID: 24335234).",

	
"SETBP1":	"SETBP1 mutations can be seen in various myeloid neoplasms, including atypical BCR-ABL1-negative CML, where they can provide supportive evidence for this diagnosis per the WHO classification. SETBP1 mutations are associated with a poor prognosis in CMML. SETBP1 mutations are associated with disease progression in MDS. SETBP1 mutations can be seen in various myeloid neoplasms, and are associated with disease progression in MDS and a poor prognosis in CMML. SETBP1, located on 18q21.1, encodes for a protein that binds the SET nuclear oncogene, which is involved in DNA replication. SETBP1 mutations are found in 30% of juvenile myelomonocytic leukemia (PMID: 26457647); 24% of atypical chronic myeloid leukemia, where they provide supportive evidence for the diagnosis in the 2016 revision of the WHO classification (PMID: 23222956, PMID: 27069254); 13% of refractory anemia with ring sideroblasts associated with marked thrombocytosis (PMID: 26874914), 9% of chronic myelomonocytic leukemia (PMID: 27385790), 3% of primary myelofibrosis (PMID: 23558523), 2% of essential thrombocythemia, 2% of polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30), 2% of myelodysplastic syndrome, and 2% of secondary acute myeloid leukemia (PMID: 23443343). SETBP1 mutations are associated with an unfavorable prognosis in chronic myelomonocytic leukemia (PMID: 27385790), juvenile myelomonocytic leukemia (PMID: 26457647), atypical chronic myeloid leukemia (PMID: 23222956), and refractory anemia with ring sideroblasts associated with marked thrombocytosis (PMID: 26874914); and disease progression in myelodysplastic syndrome.",	
"SETD2":	"SETD2, located on 3p21.31, encodes for a histone methyltransferase that is specific for lysine-36 of histone H3, and methylation of this residue is associated with active chromatin. SETD2 mutations or deletions are found in 93% of type II enteropathy-associated T-cell lymphoma (PMID: 26854024), 25% of hepatosplenic T-cell lymphoma (PMID: 28122867), 22% of MLL-rearranged acute leukemia but 5% of acute leukemia without MLL rearrangements (PMID: 22237106), 8% of pediatric early T-cell precursor acute lymphoblastic leukemia (PMID: 22237106); 7% of chronic lymphocytic leukemia (PMID: 27282254), where they are associated with loss of TP53, genomic complexity and chromothripsis; and 3% of adult T-cell leukemia/lymphoma (PMID: 26437031).",	
"SF3B1":	"In a patient with AML, mutation in SF3B1 is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, SF3B1 alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival.SF3B1, located on 2q33.1, encodes for the subunit 1 of the splicing factor 3b protein complex, which is involved in RNA splicing. SF3B1 mutations are found in 5% of chronic lymphocytic leukemia at diagnosis, 17% of fludarabine refractory chronic lymphocytic leukemia, and are associated with 11q deletion and an unfavorable prognosis (PMID: 22039264, PMID: 22150006, and PMID: 22158541). SF3B1 mutations are associated with a poor prognosis in ET. SF3B1, located on 2q33.1, encodes for the subunit 1 of the splicing factor 3b protein complex, which is involved in RNA splicing. SF3B1 mutations are found in 5% of chronic lymphocytic leukemia at diagnosis, 17% of fludarabine refractory chronic lymphocytic leukemia, and are associated with 11q deletion and an unfavorable prognosis (PMID: 22039264, PMID: 22150006, and PMID: 22158541). SF3B1 mutations are associated with a favorable prognosis in MDS and are strongly associated with ring sideroblasts. SF3B1 mutations can be seen in various myeloid neoplasms and are strongly associated with ring sideroblasts. SF3B1 mutations are associated with a favorable prognosis in MDS and a poor prognosis in AML and ET. SF3B1, located on 2q33.1, encodes for the subunit 1 of the splicing factor 3b protein complex, which is involved in RNA splicing. SF3B1 mutations are found in 85% of refractory anemia with ring sideroblasts associated with marked thrombocytosis (PMID: 26874914), 33% of myelodysplastic syndrome overall but 82% of subtypes with increased ring sideroblasts (PMID: 24220272), 12% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 7% of primary myelofibrosis (PMID: 24496303), 6% of chronic myelomonocytic leukemia (PMID: 27385790), 3% of acute myeloid leukemia (PMID: 27288520), 5% of essential thrombocythemia, and 3% of polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30).  SF3B1 mutations are associated with a favorable prognosis in myelodysplastic syndrome, and are highly predictive for the presence of ring sideroblasts (PMID: 21995386, PMID: 25957392). SF3B1 mutations are also reported to be highly specific for secondary acute myeloid leukemia, and may also be helpful in identifying a subset of elderly patients with de novo acute myeloid leukemia with worse clinical outcomes (PMID: 25550361). SF3B1 mutations are associated with an unfavorable prognosis in essential thrombocythemia (Tefferi A, et al. Blood Advances 2016;1:21-30). JAK2 mutations are frequently seen in MPN, while SF3B1 mutations are frequently seen in MDS with ring sideroblasts. Co-mutation of JAK2 and SF3B1 is suggestive of a MDS/MPN-RS-T.",	
"SGK1":	"SGK1, located on 6q23, encodes for a phosphatidylinositol-3-OH kinase-regulated kinase with functions including regulation of FOXO transcription factors, regulation of NF-kappaB, and negative regulation of NOTCH signaling. SGK1 mutations are found in about 50% of nodular lymphocyte predominant Hodgkin lymphoma (PMID: 26658840); as well as 17% of diffuse large B-cell lymphoma, with reported restriction to the germinal center B-cell-like subtype (PMID: 21796119). SGK1 mutations have not been found in the activated B-cell-like subtype of diffuse large B-cell lymphoma or Burkitt lymphoma.",	
"SH2B3":	"SH2B3 mutations are associated with a poor prognosis in ET. SH2B3 (also known as LNK), located on 12q24, encodes for a member of the SH2B adaptor family of proteins, which are involved in a range of signaling activities by growth factor and cytokine receptor. Philadelphia chromosome-like (Ph-like) acute lymphoblastic leukemia is a subgroup of B-cell acute lymphoblastic leukemia associated with an unfavorable prognosis. Although this subgroup is Ph-negative, there is an otherwise similar genetic to the Ph-positive acute lymphoblastic leukemia, with kinase-activating alterations being identified in 91% of patients with Ph-like acute lymphoblastic leukemia. Mutations involving SH2B3 are one of several kinase-activating lesions in Ph-like acute lymphoblastic leukemia that may be associated with response to tyrosine kinase inhibitors (PMID: 25207766). SH2B3 mutations are also found in 20% of type II enteropathy-associated T-cell lymphoma (PMID: 27600764) and 6% of pediatric early T-cell precursor acute lymphoblastic leukemia (PMID: 22237106).",	

"SMC1A":	"Mutations in SMC1A are associated with a poor prognosis in MDS, and are more frequently seen in patients with high IPSS scores and secondary AML. SMC1A, along with RAD21, SMC3, and STAG2, encode for subunits of the cohesin complex, which is involved in DNA double-strand break repair and chromatid segregation during mitosis. Cohesin complex mutations are generally mutually exclusive of each other and are found in 20% of secondary acute myeloid leukemia, 17% of higher-risk myelodysplastic syndrome, 11% of lower-risk myelodysplastic syndrome, 11% of de novo acute myeloid leukemia, 7% of myeloproliferative neoplasm, and 4% of myelodysplastic/myeloproliferative neoplasm. Cohesin complex mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, and are more frequently found in patients with high IPSS scores and secondary acute myeloid leukemia (PMID: 25006131).",	
"SMC3":	"Mutations in SMC3 are associated with a poor prognosis in MDS, and are more frequently seen in patients with high IPSS scores and secondary AML. SMC3, along with RAD21, SMC1A, and STAG2, encode for subunits of the cohesin complex, which is involved in DNA double-strand break repair and chromatid segregation during mitosis. Cohesin complex mutations are generally mutually exclusive of each other and are found in 20% of secondary acute myeloid leukemia, 17% of higher-risk myelodysplastic syndrome, 11% of lower-risk myelodysplastic syndrome, 11% of de novo acute myeloid leukemia, 7% of myeloproliferative neoplasm, and 4% of myelodysplastic/myeloproliferative neoplasm. Cohesin complex mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, and are more frequently found in patients with high IPSS scores and secondary acute myeloid leukemia (PMID: 25006131).",	
"SOCS1":	"SOCS1, located on 16p13.13, encodes for a member of the STAT-induced STAT inhibitor (SSI), also known as suppressor of cytokine signaling (SOCS), family. Inactivating SOCS1 mutations contribute to dysregulated JAK/STAT activity, and are found in 61% of classical Hodgkin lymphoma (PMID: 16532038), 56% of primary mediastinal B-cell lymphoma (PMID: 26819451), 50% of nodular lymphocyte predominant Hodgkin lymphoma (PMID: 17652621), 27% of follicular lymphoma (PMID: 19734449), and 16% of diffuse large B-cell lymphoma (PMID: 23296022). SOCS1 mutations are associated with an unfavorable prognosis in classical Hodgkin lymphoma (PMID:26336985) and a favorable prognosis in diffuse large B-cell lymphoma when considering only truncating mutations (PMID: 23296022).",	
"SPEN":	"SPEN, located on 1p36, encodes for a negative regulator of the NOTCH pathway via its interaction with RBPSUH. SPEN mutations are found in 14% of nodal marginal zone lymphoma (PMID: 27335277) and 5% of splenic marginal zone lymphoma (PMID: 22891273).",	
"SPOP":	"SPOP, located on 17q21.33, encodes for a protein that may modulate the transcriptional repression activities of death-associated protein 6 (DAXX), which interacts with histone deacetylase, core histones, and other histone-associated proteins. SPOP mutations appear to be uncommon in lymphoid neoplasms.",	
"SRSF2":	"In a patient with AML, mutation in SRSF2 is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, SRSF2 alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival. SRSF2 mutations are associated with a poor prognosis in MDS. SRSF2 mutations are associated with a poor prognosis in MPN. SRSF2 mutations can be seen in various myeloid neoplasms, and are associated with a poor prognosis in MDS, AML, PMF, and PV.SRSF2, located on 17q25.1, encodes for a member of the serine/arginine-rich family of pre-mRNA splicing factors, which constitute part of the spliceosome. SRSF2 mutations are found 39% of chronic myelomonocytic leukemia (PMID: 27385790), 17% of myelodysplastic syndrome (PMID: 24220272), 10% of acute myeloid leukemia (PMID: 27288520), 9% of primary myelofibrosis (PMID: 23619563), 8% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), and 3% of polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30). SRSF2 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, primary myelofibrosis (PMID: 23619563), polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30), and KIT D816V mutated advanced systemic mastocytosis (PMID: 26464169). Co-mutation of SRSF2 and TET2 is reported to be highly predictive of myelodysplasia with monocytosis, such as chronic myelomonocytic leukemia (PMID: 24970933). SRSF2 mutations are also reported to be highly specific for secondary acute myeloid leukemia, and may also be helpful in identifying a subset of elderly patients with de novo acute myeloid leukemia with worse clinical outcomes (PMID: 25550361). Co-mutation of SRSF2 and TET2 is reported to be highly predictive of a myeloid neoplasm characterized by myelodysplasia and monocytosis, including, but not limited to, CMML.",	
"STAG2":	"STAG2 mutations can be seen in various myeloid neoplasms, and are associated with a poor prognosis in MDS, AML, and MPN. In a patient with AML, mutation in STAG2 is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, STAG2 alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival. STAG2 mutations are associated with a poor prognosis in MDS. STAG2 mutations are associated with a poor prognosis in MPN. STAG2 mutations can be seen in various myeloid neoplasms, and are associated with a poor prognosis in MDS, AML, and MPN.STAG2, along with RAD21, SMC1A, and SMC3, encode for subunits of the cohesin complex, which is involved in DNA double-strand break repair and chromatid segregation during mitosis. Cohesin complex mutations are generally mutually exclusive of each other and are found in 20% of secondary acute myeloid leukemia, 17% of higher-risk myelodysplastic syndrome, 11% of lower-risk myelodysplastic syndrome, 11% of de novo acute myeloid leukemia, 7% of myeloproliferative neoplasm, and 4% of myelodysplastic/myeloproliferative neoplasm. Cohesin complex mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, and are more frequently found in patients with high IPSS scores and secondary acute myeloid leukemia (PMID: 25006131). STAG2 mutations are also reported to be highly specific for secondary acute myeloid leukemia, and may also be helpful in identifying a subset of elderly patients with de novo acute myeloid leukemia with worse clinical outcomes (PMID: 25550361).",
"STAT3":	"STAT3, located on 17q21.31, encodes for a member of the STAT family of proteins, which are involved in transcription activation. STAT3 mediates the expression of a variety of genes in response to cell stimuli, and thus plays a key role in many cellular processes such as cell growth and apoptosis. STAT3 mutations are found in 40% of T-cell large granular lymphocytic leukemia (PMID: 22591296), 30% of chronic lymphoproliferative disorder of NK-cells (PMID: 22859607), 10% of NK/T-cell lymphoma (PMID: 26192917), 10% of gamma-delta hepatosplenic T-cell lymphoma (PMID: 24947020), and 10% of ALK-negative anaplastic large cell lymphoma. STAT3 mutations, particularly the Y640F variant, are associated with an increased likelihood of response to immunosuppressive therapy in T-cell large granular lymphocytic leukemia (PMID: 25306898). STAT3 mutations occur in large granular lymphocytic leukemia (LGL) associated with MDS, and are associated with immune bone marrow failure.",	
"STAT5B":	"STAT5B, located on 17q21.2, encodes for a member of the STAT family of proteins, which are involved in transcription activation. STAT5B mediates the expression of a variety of genes in response to cell stimuli, and thus plays a key role in many cellular processes such as cell growth and apoptosis. STAT5B mutations are found in 63% of type II enteropathy-associated T-cell lymphoma (PMID: 26854024), 36% of T-cell prolymphocytic leukemia (PMID: 24825865), 33% of gamma-delta hepatosplenic T-cell lymphoma (PMID: 24947020), 9% of T-cell acute lymphoblastic leukemia (PMID: 24573384), and 2% of T-cell large granular lymphocytic leukemia. The clinical course of the disease in T-cell large granular lymphocytic leukemia with the N642H variant is reported to be aggressive (PMID: 23596048).",	
"STAT6":	"STAT6, located on 12q13.3, encodes for a member of the STAT family of proteins, which are involved in transcription activation. STAT6 mediates the expression of a variety of genes in response to cell stimuli, and thus plays a key role in many cellular processes such as cell growth and apoptosis. STAT6 mutations are found in 36% of primary mediastinal B-cell lymphoma (PMID: 19423726), 12% of follicular lymphoma (PMID: 28064239), and 4% of diffuse large B-cell lymphoma (PMID: 22343534).",	
"TBL1XR1":	"TBL1XR1, located on 3q26.32, encodes for a protein that has sequence similarity with members of the WD40 repeat-containing protein family, which may mediate protein-protein interactions and signal transduction. TBL1XR1 mutations are found in 19% of primary central nervous system lymphoma (PMID: 24970810), 17% of adult T-cell leukemia/lymphoma (PMID: 26437031), 14% of nodal marginal zone lymphoma (PMID: 27335277), and 10% of splenic marginal zone lymphoma (PMID: 22891273).",	
"TCF3":	"TCF3, located on 19p13.3, encores for a member of the E protein (class I) family of helix-loop-helix transcription factors, which activate transcription by binding to regulatory E-box sequences on target genes as heterodimers or homodimers, and are inhibited by heterodimerization with inhibitor of DNA-binding (class IV) helix-loop-helix proteins. E proteins play a critical role in lymphopoiesis, and are required for B and T-cell development. TCF3 mutations are found in 29% of Burkitt lymphoma and 4% of the germinal center B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 22885699).",	
"TET2":	"Somatic mutation in TET2 can be found in patients over the age of 50 without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals lack cytopenias and have only a single mutation with low allele frequency TET2, located on 4q24, encodes for a protein that plays a critical role in DNA methylation by converting methylcytosine to 5-hydroxymethylcytosine. TET2 mutations are commonly found in a variety of myeloid neoplasms. Among lymphoid neoplasms, TET2 mutations are found in 84% of angioimmunoblastic T-cell lymphoma, 49% of peripheral T-cell lymphoma not otherwise specified (PMID: 24413737), 12% of diffuse large B-cell lymphoma (PMID: 23831920), 12% of Sezary syndrome (PMID: 26551667), 8% of adult T-cell leukemia/lymphoma (PMID: 26437031), 5% of adult T-cell acute lymphoblastic leukemia (PMID: 25595890), 5% of hepatosplenic T-cell lymphoma (PMID: 28122867), and 4% of mantle cell lymphoma (PMID: 23407552).  Somatic mutations in leukemia-associated genes, particularly ASXL1, DNMT3A, or TET2, can also be found in about 10% of elderly individuals without known hematologic malignancy, a condition known as clonal hematopoiesis of indeterminate potential (CHIP). Most of these individuals, however, lack cytopenias and have only a single mutation with low allele frequency. The presence of a somatic mutation is associated with increases in the risk of developing a hematologic malignancy and in all-cause mortality, with the risk of developing a hematologic malignancy increased by a factor of nearly 50 in individuals with allele frequency of 10% or more (PMID: 25426837). Somatic mutations in leukemia-associated genes are also commonly found in patients with unexplained cytopenias who do not meet diagnostic criteria for myelodysplastic syndrome, a condition known as clonal cytopenias of undetermined significance (CCUS) (PMID: 26429975 and PMID: 26392596). TET2, located on 4q24, encodes for a protein that plays a critical role in DNA methylation by converting methylcytosine to 5-hydroxymethylcytosine. TET2 mutations are commonly found in a variety of myeloid neoplasms. Among lymphoid neoplasms, TET2 mutations are found in 84% of angioimmunoblastic T-cell lymphoma, 49% of peripheral T-cell lymphoma not otherwise specified (PMID: 24413737), 12% of diffuse large B-cell lymphoma (PMID: 23831920), 12% of Sezary syndrome (PMID: 26551667), 8% of adult T-cell leukemia/lymphoma (PMID: 26437031), 5% of adult T-cell acute lymphoblastic leukemia (PMID: 25595890), 5% of hepatosplenic T-cell lymphoma (PMID: 28122867), and 4% of mantle cell lymphoma (PMID: 23407552). TET2 mutations are associated with an increased response to hypomethylating agents when the allele frequency is >10% and ASXL1 is not mutated (PMID: 25224413). TET2 mutations are also associated with unfavorable outcomes and shorter survival after hematopoietic stem cell transplantation in patients with myelodysplastic syndrome (PMID: 25092778). Co-mutation of TET2 and SRSF2 is reported to be highly predictive of myelodysplasia with monocytosis, such as chronic myelomonocytic leukemia (PMID: 24970933). In acute myeloid leukemia with wild-type FLT3-ITD and normal karyotype or intermediate-risk cytogenetic abnormalities, TET2 mutations are associated with an unfavorable prognosis (PMID: 22417203). Co-mutation of TET2 and SRSF2 is reported to be highly predictive of a myeloid neoplasm characterized by myelodysplasia and monocytosis, including, but not limited to, CMML.",	
"TLR2":	"TLR2, located on 4q31.3, encodes for a member of the Toll-like receptor (TLR) family, which plays an important role in pathogenic recognition and activation of innate immunity. TLR2 mutations appear to be uncommon in lymphoid neoplasms, and are found in 2% of mantle cell lymphoma (PMID: 24145436), primarily those that are SOX11-negative and IGHV-mutated.",

	
"TNFAIP3":	"TNFAIP3, located on 6q23, encodes for a zinc finger protein and ubiqitin-editing enzyme that inhibits NF-kappaB activation as well as TNF-mediated apoptosis. Loss of TNFAIP3 function occurs through mutations or deletions, resulting in loss of inhibition of NF-kappaB signaling. TNFAIP3 alterations are found in 61% of primary mediastinal B-cell lymphoma (PMID: 26819451), 33% of nodular sclerosis classical Hodgkin lymphoma (PMID: 19412163), 24% of the activated B-cell-like subtype of diffuse large B-cell lymphoma (PMID: 19412164), 22% of extranodal marginal zone lymphoma of mucosa-associated lymphoid tissue (PMID: 19412163), 14% of nodal marginal zone lymphoma (PMID: 27335277), 11% of transformed follicular lymphoma (PMID: 24362818), 8% of T-cell large granular lymphocytic leukemia (PMID: 26199174), and a subset of splenic marginal zone lymphoma (7% mutation, 7% deletion) (PMID: 22891273).",	
"TNFRSF14":	"TNFRSF14, located on 1p36.32, encodes for a member of the TNF receptor superfamily, which is involved in signal transduction pathways that activate inflammatory and inhibitory T-cell immune response. TNFRSF14 mutations are found in 29% of pediatric-type nodal follicular lymphoma (PMID: 27325104), 22% of diffuse large B-cell lymphoma (PMID: 22343534); 18% of follicular lymphoma, where they are associated with an unfavorable prognosis, particularly when co-occurring with 1p36 deletions (PMID: 20884631); and 17% of nodal marginal zone lymphoma (PMID: 27335277).",	
"TP53":	"TP53 mutations are associated with a poor prognosis in AML. TP53, located on 17p13.1 , encodes for a tumor suppressor protein that responds to diverse cellular stresses by regulating expression of target genes, thereby inducing cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. Somatic TP53 alterations are frequent in most human cancers, and germline TP53 mutations predispose to a wide spectrum of early-onset cancers including Li-Fraumeni and Li-Fraumeni-like syndromes. TP53 alterations including TP53 mutations or 17p deletions are associated with an unfavorable prognosis and low response rates to chemotherapy and chemoimmunotherapy in chronic lymphocytic leukemia, and patients with TP53 alterations may potentially benefit from newer agents including BTK, BCL2, and PI3K delta inhibitors. TP53 alterations are also found in a variety of other lymphoid neoplasms, and are typically associated with an unfavorable prognosis. TP53 mutations are associated with a poor prognosis in ET. TP53 mutations are associated with a poor prognosis in MDS, and are more frequently seen with complex karyotypes and del(5q). TP53 mutations may predict resistance or relapse to lenalidomide., TP53 mutations are associated with a poor prognosis in MPN. TP53 mutations can be seen in various myeloid neoplasms, and are associated with a poor prognosis in MDS, AML, and MPN.TP53, located on 17p13.1, encodes a tumor suppressor protein that responds to diverse cellular stresses by regulating expression of target genes, thereby inducing cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. TP53 mutations are found in 8% of overall myelodysplastic syndrome (PMID: 21714648) but 20% of myelodysplastic syndrome with isolated del(5q) (PMID: 21519010), 9% of acute myeloid leukemia (PMID: 27288520), 8% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 4% of primary myelofibrosis (PMID: 22052707), 2% of essential thrombocythemia, 1% of polycythemia vera (Tefferi A, et al. Blood Advances 2016;1:21-30), and 1% chronic myelomonocytic leukemia (PMID: 23690417). TP53 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome, independent of IPSS, IPSS-R, age, and other gene mutations. When mutation status is integrated into the survival analysis by IPSS risk categories, TP53 mutations are reported to shift the survival curve of the IPSS risk category to resemble that of the next highest IPSS risk level (PMID: 21714648). Accordingly, current guidelines state that gene mutation analysis can refine the prognosis of myelodysplastic syndrome in patients risk stratified by the IPSS or IPSS-R, and may be helpful in patients predicted to have intermediate risk. TP53 mutations are independently associated with unfavorable outcomes and shorter survival after hematopoietic stem cell transplantation in patients with myelodysplastic syndrome and myelodysplastic syndrome/acute myeloid leukemia (PMID: 27601546), but an increased response to decitabine in patients with myelodysplastic syndrome or acute myeloid leukemia (PMID: 27959731). In low-risk myelodysplastic syndrome with isolated del(5q), TP53 mutations are associated with an unfavorable prognosis and decreased response to lenalidomide (PMID: 21519010). In myelodysplastic syndrome with complex karyotype, it appears that the adverse prognostic significance traditionally associated with complex karyotype may be dictated by the presence of TP53 mutation (PMID: 21714648). TP53 mutations are also associated with an unfavorable prognosis in acute myeloid leukemia (PMID: 27288520) and essential thrombocythemia (Tefferi A, et al. Blood Advances 2016;1:21-30). TP53 mutations are associated with leukemic transformation in PMF.",


"TRAF2":	"TRAF2, located on 9q34, encodes for a protein that is required for TNF-alpha-mediated activation of MAPK8/JNK and NF-kappaB. The protein complex formed by TRAF2 and TRAF1 interacts with the IAP family members, cIAP1 and cIAP2, and functions as a mediator of the anti-apoptotic signals from TNF receptors (PMID: 9774977). TRAF2 mutations are found in 6% of mantle cell lymphoma (PMID: 24362935), 3% of Waldenstrom macroglobulinemia (PMID: 24366360), and 1% of chronic lymphocytic leukemia (PMID: 26466571). TRAF2 mutations via involvement of the alternative NF-kappaB pathway may be associated with resistance to BCR inhibition (PMID: 24362935).",	
"TRAF3":	"TRAF3, located on 14q32.32, encodes for a member of the TNF receptor associated factor protein family. TRAF3 mutations are found in 4% of multiple myeloma (PMID: 26282654), 3% of Waldenstrom macroglobulinemia (PMID: 24366360), 3% of splenic marginal zone lymphoma (PMID: 22891273), and 1% of chronic lymphocytic leukemia (PMID: 26466571).",	
"U2AF1":	"In a patient with AML, mutation in U2AF1 is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, U2AF1 alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival. U2AF1 mutations are associated with a poor prognosis in ET. U2AF1 mutations are associated with a poor prognosis in MDS. U2AF1 mutations are associated with a poor prognosis in MPN. U2AF1 mutations can be seen in various myeloid neoplasms, and are associated with a poor prognosis in MDS and AML., U2AF1, located on 21q22.3, encodes for the small subunit of the U2 auxiliary factor, which is involved in RNA splicing. U2AF1 mutations are found in 16% of primary myelofibrosis (PMID: 24496303), 12% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 8% of myelodysplastic syndrome (PMID: 24220272), 4% of chronic myelomonocytic leukemia (PMID: 27385790), 4% of acute myeloid leukemia (PMID: 27288520), and 1% of essential thrombocythemia (Tefferi A, et al. Blood Advances 2016;1:21-30). U2AF1 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome and essential thrombocythemia (Tefferi A, et al. Blood Advances 2016;1:21-30), and decreased response to lenalidomide in myeloid neoplasms with and without del(5q) (PMID: 27560106). U2AF1 mutations are also reported to be highly specific for secondary acute myeloid leukemia, and may also be helpful in identifying a subset of elderly patients with de novo acute myeloid leukemia with worse clinical outcomes (PMID: 25550361). U2AF1 Q157 mutation is associated with a poor prognosis in PMF compared with U2AF1 S34 mutation or no U2AF1 mutation.",	
"UBR5":	"UBR5 encodes for a progestin-induced protein that functions as an E3 ubiquitin-protein ligase, targeting specific proteins for ubiquitin-mediated proteolysis. UBR5 is localized to chromosome 8q22, and is disrupted in a variety of neoplasms. URB5 mutations are found in 18% of mantle cell lymphoma (PMID: 23407552) and 10% of hepatosplenic T-cell lymphoma (PMID: 28122867).",	
"WT1":	"WT1, located on 11p13, encodes for a transcription factor necessary for the development of the urogenital system. The precise roles of WT1 in normal and malignant hematopoiesis remain uncertain. WT1 mutations are found in 10% of pediatric and adult T-cell acute lymphoblastic leukemia/lymphoma (PMID: 19847202 and PMID: 25595890). WT1 mutations are associated with a poor prognosis in MDS. WT1, located on 11p13, encodes for a transcription factor necessary for the development of the urogenital system. The precise roles of WT1 in normal and malignant hematopoiesis remain uncertain. WT1 mutations are found in 6% of acute myeloid leukemia overall, with higher frequencies in biallelic CEBPA mutated acute myeloid leukemia (14%). WT1 mutations are associated with an unfavorable prognosis in cytogenetically normal acute myeloid leukemia (PMID: 25110071).",	
"XPO1":	"XPO1, located on 2p15, encodes for a protein that mediates leucine-rich nuclear export signal-dependent protein transport. XPO1 mutations are found in 26% of classical Hodgkin lymphoma, 24% of primary mediastinal B-cell lymphoma (PMID: 27312795); and 3% of untreated chronic lymphocytic leukemia, where they are associated with unfavorable prognostic markers such as high CD38 and ZAP70 expression, NOTCH1 mutation, and unmutated IGHV (PMID: 24113472).",	
"ZFHX4":	"ZFHX4, located on 8q21.13, encodes for a protein that may be involved in neural and muscle differentiation. ZFHX4 mutations are found in 4% of multiple myeloma, where they are associated with an unfavorable prognosis (PMID: 26282654).",	
"ZMYM3":	"ZMYM3, located on Xq13.1, encodes for a component of histone deacetylase-containing multiprotein complexes that function to silence genes by modifying chromatin structure. ZMYM3 mutations are found in 4% of chronic lymphocytic leukemia (PMID: 22150006).",	
"ZRSR2":	"In a patient with AML, mutation in ZRSR2 is highly specific for secondary AML. In therapy related-AML and elderly de novo AML populations, ZRSR2 alterations are associated with worse clinical outcomes, including a lower complete remission rate, more frequent reinduction, and decreased event-free survival. ZRSR2 mutations are associated with a poor prognosis in MDS. ZRSR2 mutations are associated with a poor prognosis in MPN. ZRSR2 mutations can be seen in various myeloid neoplasms, and they are associated with a poor prognosis in MDS, MPN, and AML, and MPN. ZRSR2, located on Xp22.1, encodes for a protein that associates with the U2 auxiliary factor heterodimer, which is involving in splicing. ZRSR2 mutations are found in 8% of myelodysplastic syndrome (PMID: 24220272), 8% of blastic plasmacytoid dendritic cell neoplasm (PMID: 24072100), 4% of chronic myelomonocytic leukemia (PMID: 27385790), 2% of myeloproliferative neoplasm (PMID: 21909114), and 1% of acute myeloid leukemia (PMID: 27288520). ZRSR2 mutations are associated with an unfavorable prognosis in myelodysplastic syndrome. ZRSR2 mutations are also reported to be highly specific for secondary acute myeloid leukemia, and may also be helpful in identifying a subset of elderly patients with de novo acute myeloid leukemia with worse clinical outcomes (PMID: 25550361)."
};
