
<!DOCTYPE html>
<html>
<head>
  <title>FISH Probes (Summary)</title>
  <style>
    body {font: 14px Arial; line-height: 19.6px; padding: 0 15px;}
    a, a:visited {text-decoration: none;}
    a:hover {text-decoration: underline;}
    a, a:hover, a:visited, a:active {color: #0366d6;}
  </style>
  <script src="https://cdn.jsdelivr.net/npm/ideogram@1.28.0/dist/js/ideogram.min.js"></script>
<link rel="icon" type="image/x-icon" href="img/ideogram_favicon.ico">
</head>
<body>
  <h1 style="text-align: center">FISH Probes at <a target = "_blank" href = "https://neogenomics.com/client-services/neo-test-requisitions" title = "Click for Requisitions">NeoGenomics</a></h1>
  <!-- <h2 style="text-align: center">Data Visualization with IdeogramJS by Prof. Eric Weitz</h2> -->
  
 </p>
  <div id="container"><strong>Legend</strong><br><span style="color:blue">Blue</span> = Heme | <span style="color:yellowgreen">Green</span> = Solid tumor | <span style="color:red">Red</span> = Centromeric and anonymous sequence<br> Rectangle = Paraffin validated | Triangle = Fresh tissue</div>
  <script type="text/javascript">

  /**
  * This function gets genomic coordinate data for ACMG genes from NCBI EUtils.
  * EUtils docs: https://www.ncbi.nlm.nih.gov/books/NBK25500/
  * Ideogram.js then draws those genes as annotations on chromosomes.
  */
  
let centromeres = [{name: 'CEN1', chr: '1', start: 122785515, stop: 123045931}, {name: 'CEN9', chr: '9', start: 44297892, stop: 44876949 }, {name: 'CEN10', chr: '10', start: 39777874, stop: 40356932}, {name: 'CEN11', chr: '11', start: 53096456, stop: 53675515}, {name: 'CEN12', chr: '12', start: 36577136, stop: 37156196}, {name: 'CEN17', chr: '17', start: 24861155, stop: 25440216}, {name: 'CEN4', chr: '4', start: 49785892, stop: 50364954}, {name: 'CEN6', chr: '6', start: 59742696, stop: 60321759}, {name: 'CEN7', chr: '7', start: 60337915, stop: 60916979}, {name: 'CEN8', chr: '8', start: 44496972, stop: 45076037}, {name: 'XCEN', chr: 'X', start: 57813713, stop: 63146046}, {name: 'YCEN', chr: 'Y', start: 10248126, stop: 10865688}];

  function getAndDrawAcmgGenes() {

    var acmgGenes, i, annots, geneClause, geneID, geneIDs, topWeight,
        eutils, esearch, esummary, url, defaultParams, centromeres, anonymous
        ideo = this;

    ideo.annotDescriptions = {annots: {}}

    geneIDs = [];
    annots = [];

    // EUtils is a web API to access NCBI data
    eutils = 'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/';
    defaultParams = '?db=gene&retmode=json&retmax=100';

    // Use ESearch to get NCBI Gene ID for gene name, e.g. BRCA1 -> 672
    esearch = eutils + 'esearch.fcgi' + defaultParams;

    // Use ESummary to get genomic coordinates for given gene IDs
    esummary = eutils + 'esummary.fcgi' + defaultParams;

    acmgGenes = [
     'ABL1', 'ASS1','BCR','ACP5','BLVRB','ALK','API2','MALT1','ATM','BCL2','BCL6','CBFB','CCND1','IGH','CKDN2A','CDKN2C','CKS1B','MYC','CRLF2','LAMP1','DEK','NUP214','DLEU1','DUSP22','IRF45','IRF43','EPOR','ETV6','RUNX1','FGFR1','FGFR3','FIP1L1','CHIC2','PDGFRA','TERT','EGR1','RPS14','MAF','MAFB','JAK2','KMT2A','NUP98','PBX1','TCF3','PDGFRB','PML','RARA','RPN1','MECOM','RUNX1T1','SEC63','MYB','SMAD6','TBL1XR1','TP63','TCL1','TNFRSF14','TP53','NF1','CDKN2A', 'ALK', 'ROS1', 'DDIT3', 'EWSR1', 'MDM2', 'MYC','PDGFB', 'SS18', 'NTRK1', 'NTRK2', 'NTRK3', 'MET', 'RET', 'PTEN', 'FIP1L1', 'TPRG1L', 'ABL2', 'GLTSCR2', 'ZNF443', 'BRAF', 'EGFR', 'FGFR2', 'MYCN', 'RREB1', 'CCND1', 'EPOR', 'ERBB2', 'CRLF2','TOP2A','SRY', 
     'BIRC3', 'TCL1A', 'NOP53', 'TCF7L1', 'TPRG1L',
    ];
// console.log(acmgGenes.length);

    // Batch request all ACMG genes
    geneClause = '(' + acmgGenes.join('[symbol] OR ') + '[symbol])';

    topWeight = 0;
    weights = [];

    url = esearch + '&term=Homo-sapiens[Organism] AND ' + geneClause;
    d3.json(url).then(function(data) {
      geneIDs = data.esearchresult.idlist;
// console.log(geneIDs);

      // Batch request genomic coordinates for all ACMG genes
      url = esummary + '&id=' + geneIDs.join(',');
      d3.json(url).then(function(data) {
        var result, gene, chr, loc, start, stop, weight;
        for (i = 0; i < geneIDs.length; i++) {
          geneID = geneIDs[i];
          result = data.result[geneID];

          console.log(result);

        if (result.chromosome === "X, Y") {
          result.chromosome = "X"
          result.summary = '<span style="color:red">' +'<em>' + "PSEUDOAUTOSOMAL" + '</em>' + '</span>' + '  ' + result.summary
        };
            
        if (result.currentid !== '' || acmgGenes.indexOf(result.name) === -1 ) {
            // currentid case occurs when one gene symbol in a taxon maps to
            // multiple gene.  It seems to be annotation-run noise.
            // result.name case occurs when gene has a non-canonical alias
            // matching the gene symbol.
            // Ignore both.
            continue;
          }
          gene = result.name;
          chr = result.chromosome;
          loc = result.locationhist[0]; // better than 'genomicinfo'
          start = loc.chrstart;
          stop = loc.chrstop;
          // weight = result.geneweight;
          summary = result.summary;
          mapLoc = result.maplocation;

          
          // This is the annotations API
          ////
          annots.push({
            name: gene, // required
            chr: chr, // required
            start: start, // required
            stop: stop, // required
            // weight: weight,  //optional
            summary: summary,
            mapLoc: mapLoc
          });       

          var clinvarText = 'GeneCards';
          var clinvarBase = 'https://www.ncbi.nlm.nih.gov/clinvar';

          var clinvarUrl =
            'https://www.genecards.org/cgi-bin/carddisp.pl' +
            '?gene=' + gene; 
          var clinvarLink =
            '<a target="_blank" href="' + clinvarUrl + '">' + clinvarText + '</a>';
          var vysisUrl = 'https://www.molecular.abbott/us/en/chromosome/';
          var metasystemsUrl = 'https://metasystems-probes.com/en/probes/?c=' + chr;
          var cytocellUrl = 'https://www.ogt.com/us/products/product-search/?filters=3417,15970,15971,15972&keywords=' + gene;
          var empireUrl = 'https://www.empiregenomics.com/fish-probes/loci-fish-probes/' + chr;
          var atlasUrl = 'http://atlasgeneticsoncology.org/Indexbychrom/idxa_' + chr + '.html';
          var omimUrl = 'https://omim.org/search?index=entry&start=1&limit=10&sort=score+desc%2C+prefix_sort+desc&search=' + gene;
          var neoUrl = 'https://neogenomics.com/test-menu?search=' + gene + '&methodology=All&diseases=All';
          var vysisLink = '<a target="_blank" href= "'+vysisUrl + ''+ chr +'">Vysis</a>';
          var metasystemsLink = '<a target="_blank" href="' + metasystemsUrl +'">Metasystems</a';
          var cytocellLink = '<a target="_blank" href="' + cytocellUrl +'">CytoCell</a>';
          var empireLink = '<a target="_blank" href="' + empireUrl + '">EmpireGenomics</a>';
          var atlasLink = '<a target="_blank" href="' + atlasUrl + '">Atlas</a>';
          var omimLink = '<a target="_blank" href="' + omimUrl + '">OMIM</a>';
          var neoLink = '<a target="_blank" style="color: #009933; font-size: 14px" href="' + neoUrl + '">NeoTests</a>';
          var fullName = result.description;
          var geneSummary = result.summary;
          var bandLoc = mapLoc;
//           var object = {
//   [{
//      "BCL2":"Lymphoma gene"
//   }, {
//     "ALK" :"CDX gene"
//   }]
// }
// console.log(object[gene]);

          // console.log(object.gene[0].significance)
          // var clinicalSig = 'object.head'+ '[' + gene + ']' +'.significance';


          // Add properties like fullName or clinvarLink for later use in hyperlinkGene
          ideo.annotDescriptions.annots[gene] = {
            fullName,
            clinvarLink,
            atlasLink,
            omimLink,
            geneSummary,
            vysisLink,
            metasystemsLink,
            cytocellLink,
            empireLink,
            neoLink,
            bandLoc
          
          }

        }        

      let solidTumor = ['DDIT3', 'EWSR1', 'MDM2', 'PDGFB', 'SS18', 'NTRK1', 'NTRK2', 'NTRK3', 'MET', 'RET', 'PTEN', 'ERBB2', 'TOP2A', 'TPRG1L', 'ABL2', 'GLTSCR2', 'ZNF443', 'BRAF', 'CDKN2A', 'EGFR', 'FGFR2', 'MYCN', 'RREB1', 'ROS1'];

      let paraffinProbes = ['TCL1', 'MYC', 'BIRC3', 'BCL2', 'TNFRSF14', 'DUSP22','TP63','KMT2A', 'ALK', 'BCL6', 'CCND1', 'IGN', 'MALT1', 'PDGFRA','FIP1L1', 'CHIC2'];

      // let phLike = ['PDGFRB', 'BCR', 'ABL1', 'JAK2', 'EPOR', 'CRLF2'];
        // This block highlights the centromeric and anonymous sequence probes.
        for (i = 0; i < annots.length; i++) {
          color = (i > 84) ? 'red' : 'blue';
          annots[i].shape = 'triangle';
          annots[i].color = color;  
      
      var x = annots[i].name;


     //This block changes default shape for paraffin validated probes
        if (paraffinProbes.includes(x)){
          annots[i].shape = "rectangle";
        }

    //This block changes default color for solid tumor probes
        if (solidTumor.includes(x)){
          annots[i].color = "yellowgreen";
          annots[i].shape = "rectangle";
        }
        // if (phLike.includes(x)){
        //   annots[i].color = "peru";
        //   annots[i].shape = "triangle";
        // }
      }  
        
centromeres = [{name: 'CEN1', chr: '1', start: 122785515, stop: 123045931}, {name: 'CEN9', chr: '9', start: 44297892, stop: 44876949 }, {name: 'CEN10', chr: '10', start: 39777874, stop: 40356932}, {name: 'CEN11', chr: '11', start: 53096456, stop: 53675515}, {name: 'CEN12', chr: '12', start: 36577136, stop: 37156196}, {name: 'CEN17', chr: '17', start: 24861155, stop: 25440216}, {name: 'CEN4', chr: '4', start: 49785892, stop: 50364954}, {name: 'CEN6', chr: '6', start: 59742696, stop: 60321759}, {name: 'CEN7', chr: '7', start: 60337915, stop: 60916979}, {name: 'CEN8', chr: '8', start: 44496972, stop: 45076037}, {name: 'XCEN', chr: 'X', start: 57813713, stop: 63146046}, {name: 'YCEN', chr: 'Y', start: 10248126, stop: 10865688}];

        anonymous = [{name: '11qter', chr: '11', start: 134550699, stop: 134650699}, {name: 'D7S2926', chr: '7', start: 105116411, stop: 105116411}, {name: 'D7S2460', chr:'7', start: 117799385, stop: 117799385}, {name: 'D20S108', chr: '20', start: 42201817, stop: 42201817}, {name: 'D5S1518E', chr: '5', start: 9461279, stop: 9461279}, {name: 'D5S1976', chr: '5', start: 9971148, stop: 9971148}, {name: 'D9S1783', chr: '9', start: 99470244, stop: 99470244}, {name: '20qter', chr:'20', start:64400000, stop: 64400000}];
  

        annots = annots.concat(anonymous, centromeres);
// console.log(annots);

        ideogram.drawAnnots(annots);

      });
    });
  }


  function enhanceTooltipContent(annots) {
    var term = '(' + annots.name + '[gene])+AND+(Homo+sapiens[orgn])';
    var url = 'https://ncbi.nlm.nih.gov/gene/?term=' + term;
    var description = this.annotDescriptions.annots[annots.name];

    // Necessary if-else since the centromeres and anonymous sequence probes are not NCBI coordinates and return undefined description variable
    // Link to Abbott molecular since most of these are Abbott probes

    if(description === undefined){    
    description = annots.name,
    term = annots.name,
    chr = annots.chr,
    URL = 'https://www.molecular.abbott/us/en/chromosome/' + chr,
    annots.displayName = '<a target="_blank" href="' + URL + '" <strong style="color:red">' + annots.name + '</strong></a>',
    description.fullName = annots.name
    }  else  {
      annots.displayName =
      '<a target="_blank" href="' + url + '">' + annots.name + '</a><br/>'+ '  ' + '<span style="color:green">' + description.bandLoc + '</span>' +'<br/>'+
      description.fullName + '<br/>' +
      description.clinvarLink + '<br/>'+
      description.atlasLink + '  ' + description.omimLink + '<br/>' +
      description.geneSummary + '<br/>' + 
      description.vysisLink + '    ' + description.metasystemsLink + '<br/>' + '  ' + description.cytocellLink + '<br/>' + '    ' +
      description.empireLink + '    ' + '<br/>' +
      description.neoLink + '<br/>';
}
    return annots
  }

  

  var d3 = Ideogram.d3;

  var config = {
    organism: 'human',
    resolution: 850,
    // showBandLabels : true,
    chrHeight: 600,
    annotationHeight: 5,
    // annotationsPath: "../../data/annotations/test.json"
    onLoad: getAndDrawAcmgGenes,
    onWillShowAnnotTooltip: enhanceTooltipContent
  
  };

  var ideogram = new Ideogram(config);

  </script>
</body>
</html>
